sql/tests: TestRandomSyntaxGeneration failed

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/6704789?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/6704789?buildTab=artifacts#/) on master @ [801bfc62afd7128be180e3396d21a1e0b2daa227](https://github.com/cockroachdb/cockroach/commits/801bfc62afd7128be180e3396d21a1e0b2daa227):

Random syntax error:

```
    rsg_test.go:847: Crash detected: server panic: driver: bad connection
```
Query:

```
        REASSIGN OWNED BY ident , SESSION_USER , EXPERIMENTAL_RELOCATE TO ident;
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-20099

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/6767318?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/6767318?buildTab=artifacts#/) on master @ [0eaeeb773474716753781289788fdd087fb9b166](https://github.com/cockroachdb/cockroach/commits/0eaeeb773474716753781289788fdd087fb9b166):


```
    rsg_test.go:823: 1m20s of 5m0s: 277226 executions, 14734 successful
    rsg_test.go:823: 1m25s of 5m0s: 294947 executions, 15612 successful
    rsg_test.go:823: 1m30s of 5m0s: 312783 executions, 16433 successful
    rsg_test.go:823: 1m35s of 5m0s: 330891 executions, 17292 successful
    rsg_test.go:823: 1m40s of 5m0s: 347970 executions, 18045 successful
    rsg_test.go:823: 1m45s of 5m0s: 366154 executions, 18862 successful
    rsg_test.go:823: 1m50s of 5m0s: 383276 executions, 19678 successful
    rsg_test.go:823: 1m55s of 5m0s: 400161 executions, 20427 successful
    rsg_test.go:823: 2m0s of 5m0s: 418142 executions, 21214 successful
    rsg_test.go:823: 2m5s of 5m0s: 434691 executions, 21942 successful
    rsg_test.go:823: 2m10s of 5m0s: 451329 executions, 22696 successful
    rsg_test.go:823: 2m15s of 5m0s: 468187 executions, 23452 successful
    rsg_test.go:823: 2m20s of 5m0s: 484339 executions, 24150 successful
    rsg_test.go:823: 2m25s of 5m0s: 501190 executions, 24931 successful
    rsg_test.go:823: 2m30s of 5m0s: 518011 executions, 25656 successful
    rsg_test.go:823: 2m35s of 5m0s: 534567 executions, 26372 successful
    rsg_test.go:823: 2m40s of 5m0s: 551348 executions, 27134 successful
    rsg_test.go:823: 2m45s of 5m0s: 567537 executions, 27853 successful
    rsg_test.go:823: 2m50s of 5m0s: 584488 executions, 28583 successful
    rsg_test.go:823: 2m55s of 5m0s: 600359 executions, 29315 successful
    rsg_test.go:823: 3m0s of 5m0s: 616395 executions, 30019 successful
    rsg_test.go:823: 3m5s of 5m0s: 631989 executions, 30696 successful
    rsg_test.go:823: 3m10s of 5m0s: 647263 executions, 31371 successful
    rsg_test.go:823: 3m15s of 5m0s: 663114 executions, 32056 successful
    rsg_test.go:823: 3m20s of 5m0s: 678868 executions, 32777 successful
    rsg_test.go:823: 3m25s of 5m0s: 694566 executions, 33451 successful
    rsg_test.go:823: 3m30s of 5m0s: 709804 executions, 34161 successful
    rsg_test.go:823: 3m35s of 5m0s: 726983 executions, 34922 successful
    rsg_test.go:823: 3m40s of 5m0s: 745333 executions, 35812 successful
    rsg_test.go:823: 3m45s of 5m0s: 764123 executions, 36572 successful
    rsg_test.go:823: 3m50s of 5m0s: 782391 executions, 37422 successful
    rsg_test.go:823: 3m55s of 5m0s: 801024 executions, 38241 successful
    rsg_test.go:823: 4m0s of 5m0s: 819333 executions, 39088 successful
    rsg_test.go:823: 4m5s of 5m0s: 837418 executions, 39957 successful
    rsg_test.go:823: 4m10s of 5m0s: 855534 executions, 40840 successful
    rsg_test.go:823: 4m15s of 5m0s: 873158 executions, 41655 successful
    rsg_test.go:823: 4m20s of 5m0s: 890902 executions, 42430 successful
    rsg_test.go:823: 4m25s of 5m0s: 909136 executions, 43253 successful
    rsg_test.go:823: 4m30s of 5m0s: 927475 executions, 44068 successful
    rsg_test.go:823: 4m35s of 5m0s: 945169 executions, 44898 successful
    rsg_test.go:823: 4m40s of 5m0s: 962892 executions, 45765 successful
    rsg_test.go:823: 4m45s of 5m0s: 980889 executions, 46585 successful
    rsg_test.go:823: 4m50s of 5m0s: 998426 executions, 47430 successful
    rsg_test.go:823: 4m55s of 5m0s: 1016492 executions, 48262 successful
    rsg_test.go:823: 5m0s of 5m0s: 1034711 executions, 49081 successful
    rsg_test.go:859: 1034713 executions, 49082 successful
    rsg_test.go:867: cannot parse output of Format: sql="BACKUP DATABASE SMALLINT TO PLACEHOLDER INCREMENTAL FROM PLACEHOLDER , NOTNULL WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP DATABASE \"smallint\" TO 'placeholder' INCREMENTAL FROM 'placeholder', 'notnull' WITH ": at or near "EOF": syntax error
    rsg_test.go:306: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration705368217
--- FAIL: TestRandomSyntaxGeneration (301.29s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7423831?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7423831?buildTab=artifacts#/) on master @ [d6ae9653b67071cca09ed296711cecc821fc284e](https://github.com/cockroachdb/cockroach/commits/d6ae9653b67071cca09ed296711cecc821fc284e):


```
    rsg_test.go:823: 1m15s of 5m0s: 247277 executions, 13367 successful
    rsg_test.go:823: 1m20s of 5m0s: 264934 executions, 14224 successful
    rsg_test.go:823: 1m25s of 5m0s: 282837 executions, 15034 successful
    rsg_test.go:823: 1m30s of 5m0s: 300167 executions, 15829 successful
    rsg_test.go:823: 1m35s of 5m0s: 317218 executions, 16645 successful
    rsg_test.go:823: 1m40s of 5m0s: 334328 executions, 17408 successful
    rsg_test.go:823: 1m45s of 5m0s: 351350 executions, 18203 successful
    rsg_test.go:823: 1m50s of 5m0s: 369102 executions, 19065 successful
    rsg_test.go:823: 1m55s of 5m0s: 385337 executions, 19862 successful
    rsg_test.go:823: 2m0s of 5m0s: 402339 executions, 20665 successful
    rsg_test.go:823: 2m5s of 5m0s: 419827 executions, 21429 successful
    rsg_test.go:823: 2m10s of 5m0s: 436835 executions, 22170 successful
    rsg_test.go:823: 2m15s of 5m0s: 453647 executions, 23029 successful
    rsg_test.go:823: 2m20s of 5m0s: 470818 executions, 23864 successful
    rsg_test.go:823: 2m25s of 5m0s: 488123 executions, 24634 successful
    rsg_test.go:823: 2m30s of 5m0s: 504700 executions, 25411 successful
    rsg_test.go:823: 2m35s of 5m0s: 521887 executions, 26210 successful
    rsg_test.go:823: 2m40s of 5m0s: 538463 executions, 26990 successful
    rsg_test.go:823: 2m45s of 5m0s: 555998 executions, 27766 successful
    rsg_test.go:823: 2m50s of 5m0s: 573758 executions, 28600 successful
    rsg_test.go:823: 2m55s of 5m0s: 590933 executions, 29429 successful
    rsg_test.go:823: 3m0s of 5m0s: 607650 executions, 30242 successful
    rsg_test.go:823: 3m5s of 5m0s: 624423 executions, 31032 successful
    rsg_test.go:823: 3m10s of 5m0s: 640979 executions, 31847 successful
    rsg_test.go:823: 3m15s of 5m0s: 657656 executions, 32679 successful
    rsg_test.go:823: 3m20s of 5m0s: 674488 executions, 33500 successful
    rsg_test.go:823: 3m25s of 5m0s: 691539 executions, 34313 successful
    rsg_test.go:823: 3m30s of 5m0s: 708724 executions, 35135 successful
    rsg_test.go:823: 3m35s of 5m0s: 725387 executions, 35968 successful
    rsg_test.go:823: 3m40s of 5m0s: 742564 executions, 36780 successful
    rsg_test.go:823: 3m45s of 5m0s: 760236 executions, 37540 successful
    rsg_test.go:823: 3m50s of 5m0s: 777557 executions, 38299 successful
    rsg_test.go:823: 3m55s of 5m0s: 794323 executions, 39065 successful
    rsg_test.go:823: 4m0s of 5m0s: 811111 executions, 39842 successful
    rsg_test.go:823: 4m5s of 5m0s: 828007 executions, 40658 successful
    rsg_test.go:823: 4m10s of 5m0s: 845254 executions, 41377 successful
    rsg_test.go:823: 4m15s of 5m0s: 861672 executions, 42106 successful
    rsg_test.go:823: 4m20s of 5m0s: 878511 executions, 42867 successful
    rsg_test.go:823: 4m25s of 5m0s: 892219 executions, 43495 successful
    rsg_test.go:823: 4m30s of 5m0s: 909277 executions, 44254 successful
    rsg_test.go:823: 4m35s of 5m0s: 926451 executions, 45036 successful
    rsg_test.go:823: 4m40s of 5m0s: 943512 executions, 45835 successful
    rsg_test.go:823: 4m45s of 5m0s: 960901 executions, 46599 successful
    rsg_test.go:823: 4m50s of 5m0s: 976460 executions, 47332 successful
    rsg_test.go:823: 4m55s of 5m0s: 993449 executions, 48088 successful
    rsg_test.go:859: 1009789 executions, 48804 successful
    rsg_test.go:867: cannot parse output of Format: sql="BACKUP INTO 'string' IN ( INVOKER ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO 'string' IN 'invoker' WITH ": at or near "EOF": syntax error
    rsg_test.go:306: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3814052040
--- FAIL: TestRandomSyntaxGeneration (301.35s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7534910?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7534910?buildTab=artifacts#/) on master @ [d98d195dfa7a3480ce3e07657aa870692fe71cea](https://github.com/cockroachdb/cockroach/commits/d98d195dfa7a3480ce3e07657aa870692fe71cea):


```
    rsg_test.go:823: 1m15s of 5m0s: 238317 executions, 12995 successful
    rsg_test.go:823: 1m20s of 5m0s: 254658 executions, 13791 successful
    rsg_test.go:823: 1m25s of 5m0s: 271176 executions, 14568 successful
    rsg_test.go:823: 1m30s of 5m0s: 287964 executions, 15325 successful
    rsg_test.go:823: 1m35s of 5m0s: 304552 executions, 16081 successful
    rsg_test.go:823: 1m40s of 5m0s: 320807 executions, 16811 successful
    rsg_test.go:823: 1m45s of 5m0s: 337237 executions, 17585 successful
    rsg_test.go:823: 1m50s of 5m0s: 352566 executions, 18305 successful
    rsg_test.go:823: 1m55s of 5m0s: 368290 executions, 19072 successful
    rsg_test.go:823: 2m0s of 5m0s: 384034 executions, 19810 successful
    rsg_test.go:823: 2m5s of 5m0s: 399968 executions, 20637 successful
    rsg_test.go:823: 2m10s of 5m0s: 415881 executions, 21380 successful
    rsg_test.go:823: 2m15s of 5m0s: 431045 executions, 22126 successful
    rsg_test.go:823: 2m20s of 5m0s: 447585 executions, 22902 successful
    rsg_test.go:823: 2m25s of 5m0s: 463652 executions, 23636 successful
    rsg_test.go:823: 2m30s of 5m0s: 479872 executions, 24385 successful
    rsg_test.go:823: 2m35s of 5m0s: 496168 executions, 25223 successful
    rsg_test.go:823: 2m40s of 5m0s: 512448 executions, 25994 successful
    rsg_test.go:823: 2m45s of 5m0s: 528873 executions, 26775 successful
    rsg_test.go:823: 2m50s of 5m0s: 545209 executions, 27585 successful
    rsg_test.go:823: 2m55s of 5m0s: 561906 executions, 28307 successful
    rsg_test.go:823: 3m0s of 5m0s: 578902 executions, 29057 successful
    rsg_test.go:823: 3m5s of 5m0s: 595305 executions, 29803 successful
    rsg_test.go:823: 3m10s of 5m0s: 611966 executions, 30526 successful
    rsg_test.go:823: 3m15s of 5m0s: 628751 executions, 31241 successful
    rsg_test.go:823: 3m20s of 5m0s: 645183 executions, 31978 successful
    rsg_test.go:823: 3m25s of 5m0s: 661808 executions, 32751 successful
    rsg_test.go:823: 3m30s of 5m0s: 678673 executions, 33491 successful
    rsg_test.go:823: 3m35s of 5m0s: 695469 executions, 34235 successful
    rsg_test.go:823: 3m40s of 5m0s: 712101 executions, 35026 successful
    rsg_test.go:823: 3m45s of 5m0s: 728660 executions, 35803 successful
    rsg_test.go:823: 3m50s of 5m0s: 745193 executions, 36616 successful
    rsg_test.go:823: 3m55s of 5m0s: 761614 executions, 37394 successful
    rsg_test.go:823: 4m0s of 5m0s: 778304 executions, 38190 successful
    rsg_test.go:823: 4m5s of 5m0s: 794483 executions, 38955 successful
    rsg_test.go:823: 4m10s of 5m0s: 810597 executions, 39719 successful
    rsg_test.go:823: 4m15s of 5m0s: 826713 executions, 40504 successful
    rsg_test.go:823: 4m20s of 5m0s: 843001 executions, 41304 successful
    rsg_test.go:823: 4m25s of 5m0s: 858834 executions, 42077 successful
    rsg_test.go:823: 4m30s of 5m0s: 874775 executions, 42801 successful
    rsg_test.go:823: 4m35s of 5m0s: 891661 executions, 43522 successful
    rsg_test.go:823: 4m40s of 5m0s: 907889 executions, 44249 successful
    rsg_test.go:823: 4m45s of 5m0s: 923911 executions, 45008 successful
    rsg_test.go:823: 4m50s of 5m0s: 938948 executions, 45692 successful
    rsg_test.go:823: 4m55s of 5m0s: 955865 executions, 46451 successful
    rsg_test.go:859: 972492 executions, 47198 successful
    rsg_test.go:867: cannot parse output of Format: sql="BACKUP INTO LATEST IN RIGHT WITH DETACHED = FALSE", formattedSQL="BACKUP INTO LATEST IN 'right' WITH ": at or near "EOF": syntax error
    rsg_test.go:306: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3495533013
--- FAIL: TestRandomSyntaxGeneration (301.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7655675?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7655675?buildTab=artifacts#/) on master @ [0d9669a8669cf5abe22b61e1f1faf9a520f2f372](https://github.com/cockroachdb/cockroach/commits/0d9669a8669cf5abe22b61e1f1faf9a520f2f372):


```
    rsg_test.go:822: 1m20s of 5m0s: 268150 executions, 14656 successful
    rsg_test.go:822: 1m25s of 5m0s: 285893 executions, 15492 successful
    rsg_test.go:822: 1m30s of 5m0s: 303733 executions, 16340 successful
    rsg_test.go:822: 1m35s of 5m0s: 322166 executions, 17171 successful
    rsg_test.go:822: 1m40s of 5m0s: 339219 executions, 17996 successful
    rsg_test.go:822: 1m45s of 5m0s: 357389 executions, 18862 successful
    rsg_test.go:822: 1m50s of 5m0s: 374877 executions, 19701 successful
    rsg_test.go:822: 1m55s of 5m0s: 391987 executions, 20524 successful
    rsg_test.go:822: 2m0s of 5m0s: 409379 executions, 21347 successful
    rsg_test.go:822: 2m5s of 5m0s: 426923 executions, 22165 successful
    rsg_test.go:822: 2m10s of 5m0s: 444036 executions, 22968 successful
    rsg_test.go:822: 2m15s of 5m0s: 462053 executions, 23800 successful
    rsg_test.go:822: 2m20s of 5m0s: 479924 executions, 24611 successful
    rsg_test.go:822: 2m25s of 5m0s: 497472 executions, 25432 successful
    rsg_test.go:822: 2m30s of 5m0s: 515664 executions, 26285 successful
    rsg_test.go:822: 2m35s of 5m0s: 533300 executions, 27150 successful
    rsg_test.go:822: 2m40s of 5m0s: 550667 executions, 27994 successful
    rsg_test.go:822: 2m45s of 5m0s: 569101 executions, 28833 successful
    rsg_test.go:822: 2m50s of 5m0s: 587044 executions, 29660 successful
    rsg_test.go:822: 2m55s of 5m0s: 605419 executions, 30486 successful
    rsg_test.go:822: 3m0s of 5m0s: 622981 executions, 31319 successful
    rsg_test.go:822: 3m5s of 5m0s: 640352 executions, 32126 successful
    rsg_test.go:822: 3m10s of 5m0s: 658086 executions, 32935 successful
    rsg_test.go:822: 3m15s of 5m0s: 674597 executions, 33654 successful
    rsg_test.go:822: 3m20s of 5m0s: 691482 executions, 34461 successful
    rsg_test.go:822: 3m25s of 5m0s: 709091 executions, 35248 successful
    rsg_test.go:822: 3m30s of 5m0s: 726208 executions, 36061 successful
    rsg_test.go:822: 3m35s of 5m0s: 743660 executions, 36870 successful
    rsg_test.go:822: 3m40s of 5m0s: 761316 executions, 37723 successful
    rsg_test.go:822: 3m45s of 5m0s: 779373 executions, 38552 successful
    rsg_test.go:822: 3m50s of 5m0s: 797339 executions, 39454 successful
    rsg_test.go:822: 3m55s of 5m0s: 815515 executions, 40269 successful
    rsg_test.go:822: 4m0s of 5m0s: 832741 executions, 41085 successful
    rsg_test.go:822: 4m5s of 5m0s: 850833 executions, 41909 successful
    rsg_test.go:822: 4m10s of 5m0s: 868023 executions, 42706 successful
    rsg_test.go:822: 4m15s of 5m0s: 885450 executions, 43560 successful
    rsg_test.go:822: 4m20s of 5m0s: 902866 executions, 44409 successful
    rsg_test.go:822: 4m25s of 5m0s: 921063 executions, 45226 successful
    rsg_test.go:822: 4m30s of 5m0s: 938862 executions, 46103 successful
    rsg_test.go:822: 4m35s of 5m0s: 956950 executions, 46953 successful
    rsg_test.go:822: 4m40s of 5m0s: 974804 executions, 47768 successful
    rsg_test.go:822: 4m45s of 5m0s: 993422 executions, 48606 successful
    rsg_test.go:822: 4m50s of 5m0s: 1011869 executions, 49412 successful
    rsg_test.go:822: 4m55s of 5m0s: 1030214 executions, 50266 successful
    rsg_test.go:822: 5m0s of 5m0s: 1048217 executions, 51103 successful
    rsg_test.go:858: 1048218 executions, 51103 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP VIRTUAL , FAMILY , FAMILY . CONCURRENTLY . * , NOTHING_AFTER_RETURNING . ident INTO ( FAMILY , PLACEHOLDER , PLACEHOLDER , PLACEHOLDER , FAMILY ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TABLE \"virtual\", \"family\", \"family\".concurrently.*, \"nothing_after_returning\".ident INTO ('family', 'placeholder', 'placeholder', 'placeholder', 'family') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration1481170529
--- FAIL: TestRandomSyntaxGeneration (301.34s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7672301?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7672301?buildTab=artifacts#/) on master @ [39250f643837156c8fb75b0ced07ef8a99c14185](https://github.com/cockroachdb/cockroach/commits/39250f643837156c8fb75b0ced07ef8a99c14185):


```
    rsg_test.go:822: 1m20s of 5m0s: 261433 executions, 14081 successful
    rsg_test.go:822: 1m25s of 5m0s: 277705 executions, 14840 successful
    rsg_test.go:822: 1m30s of 5m0s: 293396 executions, 15684 successful
    rsg_test.go:822: 1m35s of 5m0s: 309998 executions, 16490 successful
    rsg_test.go:822: 1m40s of 5m0s: 327067 executions, 17251 successful
    rsg_test.go:822: 1m45s of 5m0s: 344131 executions, 18055 successful
    rsg_test.go:822: 1m50s of 5m0s: 360843 executions, 18922 successful
    rsg_test.go:822: 1m55s of 5m0s: 377728 executions, 19674 successful
    rsg_test.go:822: 2m0s of 5m0s: 394454 executions, 20467 successful
    rsg_test.go:822: 2m5s of 5m0s: 411045 executions, 21292 successful
    rsg_test.go:822: 2m10s of 5m0s: 428240 executions, 22030 successful
    rsg_test.go:822: 2m15s of 5m0s: 444421 executions, 22843 successful
    rsg_test.go:822: 2m20s of 5m0s: 460997 executions, 23683 successful
    rsg_test.go:822: 2m25s of 5m0s: 478306 executions, 24520 successful
    rsg_test.go:822: 2m30s of 5m0s: 495856 executions, 25312 successful
    rsg_test.go:822: 2m35s of 5m0s: 513362 executions, 26121 successful
    rsg_test.go:822: 2m40s of 5m0s: 530678 executions, 26881 successful
    rsg_test.go:822: 2m45s of 5m0s: 546828 executions, 27626 successful
    rsg_test.go:822: 2m50s of 5m0s: 563366 executions, 28440 successful
    rsg_test.go:822: 2m55s of 5m0s: 579687 executions, 29227 successful
    rsg_test.go:822: 3m0s of 5m0s: 596464 executions, 29952 successful
    rsg_test.go:822: 3m5s of 5m0s: 613600 executions, 30748 successful
    rsg_test.go:822: 3m10s of 5m0s: 630552 executions, 31593 successful
    rsg_test.go:822: 3m15s of 5m0s: 647065 executions, 32367 successful
    rsg_test.go:822: 3m20s of 5m0s: 663484 executions, 33176 successful
    rsg_test.go:822: 3m25s of 5m0s: 679367 executions, 33952 successful
    rsg_test.go:822: 3m30s of 5m0s: 696299 executions, 34765 successful
    rsg_test.go:822: 3m35s of 5m0s: 713610 executions, 35558 successful
    rsg_test.go:822: 3m40s of 5m0s: 730465 executions, 36401 successful
    rsg_test.go:822: 3m45s of 5m0s: 747224 executions, 37198 successful
    rsg_test.go:822: 3m50s of 5m0s: 763926 executions, 37994 successful
    rsg_test.go:822: 3m55s of 5m0s: 779778 executions, 38715 successful
    rsg_test.go:822: 4m0s of 5m0s: 796768 executions, 39491 successful
    rsg_test.go:822: 4m5s of 5m0s: 813519 executions, 40288 successful
    rsg_test.go:822: 4m10s of 5m0s: 830110 executions, 41104 successful
    rsg_test.go:822: 4m15s of 5m0s: 846828 executions, 41935 successful
    rsg_test.go:822: 4m20s of 5m0s: 863478 executions, 42697 successful
    rsg_test.go:822: 4m25s of 5m0s: 879953 executions, 43465 successful
    rsg_test.go:822: 4m30s of 5m0s: 896408 executions, 44199 successful
    rsg_test.go:822: 4m35s of 5m0s: 913196 executions, 44978 successful
    rsg_test.go:822: 4m40s of 5m0s: 929855 executions, 45685 successful
    rsg_test.go:822: 4m45s of 5m0s: 945551 executions, 46448 successful
    rsg_test.go:822: 4m50s of 5m0s: 962308 executions, 47232 successful
    rsg_test.go:822: 4m55s of 5m0s: 978223 executions, 47979 successful
    rsg_test.go:822: 5m0s of 5m0s: 993676 executions, 48728 successful
    rsg_test.go:858: 993677 executions, 48728 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP VIRTUAL , FAMILY , * , FAMILY , NOTHING_AFTER_RETURNING , INDEX_BEFORE_PAREN INTO ( PLACEHOLDER ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TABLE \"virtual\", \"family\", *, \"family\", \"nothing_after_returning\", \"index_before_paren\" INTO 'placeholder' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3896017096
--- FAIL: TestRandomSyntaxGeneration (301.52s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7685369?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7685369?buildTab=artifacts#/) on master @ [b5be006bedd7d3cedc3fb3d2248df168e3d64be2](https://github.com/cockroachdb/cockroach/commits/b5be006bedd7d3cedc3fb3d2248df168e3d64be2):


```
    rsg_test.go:822: 1m20s of 5m0s: 278508 executions, 15067 successful
    rsg_test.go:822: 1m25s of 5m0s: 296476 executions, 15971 successful
    rsg_test.go:822: 1m30s of 5m0s: 315136 executions, 16848 successful
    rsg_test.go:822: 1m35s of 5m0s: 333018 executions, 17674 successful
    rsg_test.go:822: 1m40s of 5m0s: 351994 executions, 18604 successful
    rsg_test.go:822: 1m45s of 5m0s: 370657 executions, 19528 successful
    rsg_test.go:822: 1m50s of 5m0s: 389260 executions, 20411 successful
    rsg_test.go:822: 1m55s of 5m0s: 407776 executions, 21293 successful
    rsg_test.go:822: 2m0s of 5m0s: 425420 executions, 22150 successful
    rsg_test.go:822: 2m5s of 5m0s: 443264 executions, 23027 successful
    rsg_test.go:822: 2m10s of 5m0s: 461340 executions, 23825 successful
    rsg_test.go:822: 2m15s of 5m0s: 479668 executions, 24641 successful
    rsg_test.go:822: 2m20s of 5m0s: 498054 executions, 25512 successful
    rsg_test.go:822: 2m25s of 5m0s: 517279 executions, 26419 successful
    rsg_test.go:822: 2m30s of 5m0s: 535741 executions, 27350 successful
    rsg_test.go:822: 2m35s of 5m0s: 554517 executions, 28255 successful
    rsg_test.go:822: 2m40s of 5m0s: 573624 executions, 29180 successful
    rsg_test.go:822: 2m45s of 5m0s: 592287 executions, 30063 successful
    rsg_test.go:822: 2m50s of 5m0s: 610703 executions, 30928 successful
    rsg_test.go:822: 2m55s of 5m0s: 629377 executions, 31755 successful
    rsg_test.go:822: 3m0s of 5m0s: 647400 executions, 32558 successful
    rsg_test.go:822: 3m5s of 5m0s: 665647 executions, 33416 successful
    rsg_test.go:822: 3m10s of 5m0s: 684130 executions, 34345 successful
    rsg_test.go:822: 3m15s of 5m0s: 702701 executions, 35238 successful
    rsg_test.go:822: 3m20s of 5m0s: 720738 executions, 36085 successful
    rsg_test.go:822: 3m25s of 5m0s: 738929 executions, 36959 successful
    rsg_test.go:822: 3m30s of 5m0s: 756948 executions, 37836 successful
    rsg_test.go:822: 3m35s of 5m0s: 774267 executions, 38695 successful
    rsg_test.go:822: 3m40s of 5m0s: 792953 executions, 39602 successful
    rsg_test.go:822: 3m45s of 5m0s: 811767 executions, 40455 successful
    rsg_test.go:822: 3m50s of 5m0s: 830386 executions, 41339 successful
    rsg_test.go:822: 3m55s of 5m0s: 849898 executions, 42233 successful
    rsg_test.go:822: 4m0s of 5m0s: 867578 executions, 43059 successful
    rsg_test.go:822: 4m5s of 5m0s: 885890 executions, 43925 successful
    rsg_test.go:822: 4m10s of 5m0s: 904551 executions, 44851 successful
    rsg_test.go:822: 4m15s of 5m0s: 922954 executions, 45650 successful
    rsg_test.go:822: 4m20s of 5m0s: 940528 executions, 46516 successful
    rsg_test.go:822: 4m25s of 5m0s: 958657 executions, 47334 successful
    rsg_test.go:822: 4m30s of 5m0s: 977400 executions, 48154 successful
    rsg_test.go:822: 4m35s of 5m0s: 995931 executions, 49008 successful
    rsg_test.go:822: 4m40s of 5m0s: 1013840 executions, 49913 successful
    rsg_test.go:822: 4m45s of 5m0s: 1032814 executions, 50799 successful
    rsg_test.go:822: 4m50s of 5m0s: 1051652 executions, 51679 successful
    rsg_test.go:822: 4m55s of 5m0s: 1070624 executions, 52585 successful
    rsg_test.go:822: 5m0s of 5m0s: 1088855 executions, 53443 successful
    rsg_test.go:858: 1088856 executions, 53443 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP FAMILY . CAST , FAMILY , GREATEST TO ( TREAT , 'string' , 'string' , PLACEHOLDER , PLACEHOLDER , FAMILY ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TABLE \"family\".cast, \"family\", \"greatest\" TO ('treat', 'string', 'string', 'placeholder', 'placeholder', 'family') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3344489990
--- FAIL: TestRandomSyntaxGeneration (301.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7799013?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7799013?buildTab=artifacts#/) on master @ [e7b15ebaed9c14668ade0a7827a5525aedef1ab0](https://github.com/cockroachdb/cockroach/commits/e7b15ebaed9c14668ade0a7827a5525aedef1ab0):


```
    rsg_test.go:822: 1m15s of 5m0s: 212894 executions, 11830 successful
    rsg_test.go:822: 1m20s of 5m0s: 227396 executions, 12570 successful
    rsg_test.go:822: 1m25s of 5m0s: 241982 executions, 13282 successful
    rsg_test.go:822: 1m30s of 5m0s: 256254 executions, 13982 successful
    rsg_test.go:822: 1m35s of 5m0s: 271252 executions, 14668 successful
    rsg_test.go:822: 1m40s of 5m0s: 285984 executions, 15302 successful
    rsg_test.go:822: 1m45s of 5m0s: 299961 executions, 15983 successful
    rsg_test.go:822: 1m50s of 5m0s: 314476 executions, 16628 successful
    rsg_test.go:822: 1m55s of 5m0s: 328599 executions, 17323 successful
    rsg_test.go:822: 2m0s of 5m0s: 343150 executions, 18070 successful
    rsg_test.go:822: 2m5s of 5m0s: 357927 executions, 18706 successful
    rsg_test.go:822: 2m10s of 5m0s: 372538 executions, 19366 successful
    rsg_test.go:822: 2m15s of 5m0s: 386803 executions, 20028 successful
    rsg_test.go:822: 2m20s of 5m0s: 400929 executions, 20727 successful
    rsg_test.go:822: 2m25s of 5m0s: 415574 executions, 21437 successful
    rsg_test.go:822: 2m30s of 5m0s: 429644 executions, 22053 successful
    rsg_test.go:822: 2m35s of 5m0s: 444030 executions, 22674 successful
    rsg_test.go:822: 2m40s of 5m0s: 458177 executions, 23313 successful
    rsg_test.go:822: 2m45s of 5m0s: 471885 executions, 23981 successful
    rsg_test.go:822: 2m50s of 5m0s: 486744 executions, 24670 successful
    rsg_test.go:822: 2m55s of 5m0s: 501163 executions, 25344 successful
    rsg_test.go:822: 3m0s of 5m0s: 516073 executions, 25996 successful
    rsg_test.go:822: 3m5s of 5m0s: 530229 executions, 26652 successful
    rsg_test.go:822: 3m10s of 5m0s: 544919 executions, 27371 successful
    rsg_test.go:822: 3m15s of 5m0s: 559560 executions, 28023 successful
    rsg_test.go:822: 3m20s of 5m0s: 574336 executions, 28750 successful
    rsg_test.go:822: 3m25s of 5m0s: 589006 executions, 29473 successful
    rsg_test.go:822: 3m30s of 5m0s: 602957 executions, 30157 successful
    rsg_test.go:822: 3m35s of 5m0s: 617711 executions, 30856 successful
    rsg_test.go:822: 3m40s of 5m0s: 632053 executions, 31526 successful
    rsg_test.go:822: 3m45s of 5m0s: 646667 executions, 32181 successful
    rsg_test.go:822: 3m50s of 5m0s: 661201 executions, 32814 successful
    rsg_test.go:822: 3m55s of 5m0s: 675519 executions, 33497 successful
    rsg_test.go:822: 4m0s of 5m0s: 689092 executions, 34152 successful
    rsg_test.go:822: 4m5s of 5m0s: 702619 executions, 34814 successful
    rsg_test.go:822: 4m10s of 5m0s: 716893 executions, 35472 successful
    rsg_test.go:822: 4m15s of 5m0s: 731469 executions, 36169 successful
    rsg_test.go:822: 4m20s of 5m0s: 746251 executions, 36906 successful
    rsg_test.go:822: 4m25s of 5m0s: 761501 executions, 37603 successful
    rsg_test.go:822: 4m30s of 5m0s: 776585 executions, 38235 successful
    rsg_test.go:822: 4m35s of 5m0s: 791457 executions, 38963 successful
    rsg_test.go:822: 4m40s of 5m0s: 805706 executions, 39636 successful
    rsg_test.go:822: 4m45s of 5m0s: 819746 executions, 40287 successful
    rsg_test.go:822: 4m50s of 5m0s: 833950 executions, 40956 successful
    rsg_test.go:822: 4m55s of 5m0s: 847950 executions, 41620 successful
    rsg_test.go:858: 861183 executions, 42226 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP DATABASE NOVIEWACTIVITYREDACTED INTO LATEST IN PLACEHOLDER WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP DATABASE noviewactivityredacted INTO LATEST IN 'placeholder' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2979838126
--- FAIL: TestRandomSyntaxGeneration (301.52s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7815203?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7815203?buildTab=artifacts#/) on master @ [9c21578450e395c83a1dc0df7090296fef06e006](https://github.com/cockroachdb/cockroach/commits/9c21578450e395c83a1dc0df7090296fef06e006):


```
    rsg_test.go:822: 1m15s of 5m0s: 215307 executions, 12007 successful
    rsg_test.go:822: 1m20s of 5m0s: 230609 executions, 12755 successful
    rsg_test.go:822: 1m25s of 5m0s: 245707 executions, 13488 successful
    rsg_test.go:822: 1m30s of 5m0s: 260874 executions, 14223 successful
    rsg_test.go:822: 1m35s of 5m0s: 275441 executions, 14987 successful
    rsg_test.go:822: 1m40s of 5m0s: 289508 executions, 15685 successful
    rsg_test.go:822: 1m45s of 5m0s: 304311 executions, 16410 successful
    rsg_test.go:822: 1m50s of 5m0s: 319457 executions, 17185 successful
    rsg_test.go:822: 1m55s of 5m0s: 334371 executions, 17880 successful
    rsg_test.go:822: 2m0s of 5m0s: 349023 executions, 18655 successful
    rsg_test.go:822: 2m5s of 5m0s: 364040 executions, 19374 successful
    rsg_test.go:822: 2m10s of 5m0s: 379000 executions, 20088 successful
    rsg_test.go:822: 2m15s of 5m0s: 393774 executions, 20736 successful
    rsg_test.go:822: 2m20s of 5m0s: 408843 executions, 21482 successful
    rsg_test.go:822: 2m25s of 5m0s: 423548 executions, 22179 successful
    rsg_test.go:822: 2m30s of 5m0s: 436859 executions, 22824 successful
    rsg_test.go:822: 2m35s of 5m0s: 451152 executions, 23559 successful
    rsg_test.go:822: 2m40s of 5m0s: 465760 executions, 24237 successful
    rsg_test.go:822: 2m45s of 5m0s: 480743 executions, 24933 successful
    rsg_test.go:822: 2m50s of 5m0s: 495519 executions, 25667 successful
    rsg_test.go:822: 2m55s of 5m0s: 510127 executions, 26393 successful
    rsg_test.go:822: 3m0s of 5m0s: 524228 executions, 27092 successful
    rsg_test.go:822: 3m5s of 5m0s: 538237 executions, 27769 successful
    rsg_test.go:822: 3m10s of 5m0s: 552719 executions, 28433 successful
    rsg_test.go:822: 3m15s of 5m0s: 565738 executions, 29040 successful
    rsg_test.go:822: 3m20s of 5m0s: 580128 executions, 29701 successful
    rsg_test.go:822: 3m25s of 5m0s: 595253 executions, 30426 successful
    rsg_test.go:822: 3m30s of 5m0s: 610727 executions, 31133 successful
    rsg_test.go:822: 3m35s of 5m0s: 625109 executions, 31807 successful
    rsg_test.go:822: 3m40s of 5m0s: 639756 executions, 32507 successful
    rsg_test.go:822: 3m45s of 5m0s: 653913 executions, 33131 successful
    rsg_test.go:822: 3m50s of 5m0s: 668295 executions, 33827 successful
    rsg_test.go:822: 3m55s of 5m0s: 683522 executions, 34555 successful
    rsg_test.go:822: 4m0s of 5m0s: 699226 executions, 35309 successful
    rsg_test.go:822: 4m5s of 5m0s: 714364 executions, 36068 successful
    rsg_test.go:822: 4m10s of 5m0s: 729630 executions, 36777 successful
    rsg_test.go:822: 4m15s of 5m0s: 744870 executions, 37518 successful
    rsg_test.go:822: 4m20s of 5m0s: 759641 executions, 38226 successful
    rsg_test.go:822: 4m25s of 5m0s: 774446 executions, 38943 successful
    rsg_test.go:822: 4m30s of 5m0s: 789425 executions, 39635 successful
    rsg_test.go:822: 4m35s of 5m0s: 803888 executions, 40304 successful
    rsg_test.go:822: 4m40s of 5m0s: 818893 executions, 40972 successful
    rsg_test.go:822: 4m45s of 5m0s: 833038 executions, 41604 successful
    rsg_test.go:822: 4m50s of 5m0s: 846358 executions, 42270 successful
    rsg_test.go:822: 4m55s of 5m0s: 859059 executions, 42871 successful
    rsg_test.go:858: 873575 executions, 43487 successful
    rsg_test.go:866: cannot parse output of Format: sql="PREPARE BUNDLE AS BACKUP INDEX_AFTER_ORDER_BY_BEFORE_AT . ident , CHARACTER . * INTO LATEST IN MULTILINESTRING WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="PREPARE bundle AS BACKUP TABLE \"index_after_order_by_before_at\".ident, \"character\".* INTO LATEST IN 'multilinestring' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration1636415489
--- FAIL: TestRandomSyntaxGeneration (301.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7834216?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7834216?buildTab=artifacts#/) on master @ [8a5cb5109d6b7e122e996a44f940bfd61698cd81](https://github.com/cockroachdb/cockroach/commits/8a5cb5109d6b7e122e996a44f940bfd61698cd81):


```
    rsg_test.go:822: 1m20s of 5m0s: 225897 executions, 12321 successful
    rsg_test.go:822: 1m25s of 5m0s: 240491 executions, 13040 successful
    rsg_test.go:822: 1m30s of 5m0s: 255063 executions, 13761 successful
    rsg_test.go:822: 1m35s of 5m0s: 269541 executions, 14477 successful
    rsg_test.go:822: 1m40s of 5m0s: 283348 executions, 15159 successful
    rsg_test.go:822: 1m45s of 5m0s: 297538 executions, 15830 successful
    rsg_test.go:822: 1m50s of 5m0s: 312544 executions, 16581 successful
    rsg_test.go:822: 1m55s of 5m0s: 327520 executions, 17307 successful
    rsg_test.go:822: 2m0s of 5m0s: 342443 executions, 18021 successful
    rsg_test.go:822: 2m5s of 5m0s: 357659 executions, 18754 successful
    rsg_test.go:822: 2m10s of 5m0s: 372535 executions, 19463 successful
    rsg_test.go:822: 2m15s of 5m0s: 387806 executions, 20158 successful
    rsg_test.go:822: 2m20s of 5m0s: 403004 executions, 20909 successful
    rsg_test.go:822: 2m25s of 5m0s: 418272 executions, 21624 successful
    rsg_test.go:822: 2m30s of 5m0s: 432697 executions, 22322 successful
    rsg_test.go:822: 2m35s of 5m0s: 447294 executions, 23074 successful
    rsg_test.go:822: 2m40s of 5m0s: 462348 executions, 23813 successful
    rsg_test.go:822: 2m45s of 5m0s: 477239 executions, 24467 successful
    rsg_test.go:822: 2m50s of 5m0s: 491921 executions, 25202 successful
    rsg_test.go:822: 2m55s of 5m0s: 506808 executions, 25855 successful
    rsg_test.go:822: 3m0s of 5m0s: 521462 executions, 26549 successful
    rsg_test.go:822: 3m5s of 5m0s: 535924 executions, 27233 successful
    rsg_test.go:822: 3m10s of 5m0s: 551156 executions, 27958 successful
    rsg_test.go:822: 3m15s of 5m0s: 566011 executions, 28634 successful
    rsg_test.go:822: 3m20s of 5m0s: 580182 executions, 29295 successful
    rsg_test.go:822: 3m25s of 5m0s: 594616 executions, 29987 successful
    rsg_test.go:822: 3m30s of 5m0s: 608824 executions, 30689 successful
    rsg_test.go:822: 3m35s of 5m0s: 623688 executions, 31406 successful
    rsg_test.go:822: 3m40s of 5m0s: 638565 executions, 32080 successful
    rsg_test.go:822: 3m45s of 5m0s: 653368 executions, 32730 successful
    rsg_test.go:822: 3m50s of 5m0s: 667749 executions, 33425 successful
    rsg_test.go:822: 3m55s of 5m0s: 683029 executions, 34197 successful
    rsg_test.go:822: 4m0s of 5m0s: 698276 executions, 34935 successful
    rsg_test.go:822: 4m5s of 5m0s: 713189 executions, 35686 successful
    rsg_test.go:822: 4m10s of 5m0s: 728317 executions, 36377 successful
    rsg_test.go:822: 4m15s of 5m0s: 742556 executions, 37047 successful
    rsg_test.go:822: 4m20s of 5m0s: 757628 executions, 37771 successful
    rsg_test.go:822: 4m25s of 5m0s: 772127 executions, 38486 successful
    rsg_test.go:822: 4m30s of 5m0s: 786809 executions, 39158 successful
    rsg_test.go:822: 4m35s of 5m0s: 801452 executions, 39810 successful
    rsg_test.go:822: 4m40s of 5m0s: 815841 executions, 40478 successful
    rsg_test.go:822: 4m45s of 5m0s: 830580 executions, 41152 successful
    rsg_test.go:822: 4m50s of 5m0s: 844497 executions, 41830 successful
    rsg_test.go:822: 4m55s of 5m0s: 858178 executions, 42510 successful
    rsg_test.go:822: 5m0s of 5m0s: 871760 executions, 43184 successful
    rsg_test.go:858: 871761 executions, 43184 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INDEX_AFTER_ORDER_BY_BEFORE_AT . POINTZM . * TO 'string' INCREMENTAL FROM VARBIT , PLACEHOLDER WITH DETACHED = FALSE", formattedSQL="BACKUP TABLE \"index_after_order_by_before_at\".pointzm.* TO 'string' INCREMENTAL FROM 'varbit', 'placeholder' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration835683825
--- FAIL: TestRandomSyntaxGeneration (301.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7867058?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7867058?buildTab=artifacts#/) on master @ [8165e3974c10e88b6ae11c6255872ea16f3a67e3](https://github.com/cockroachdb/cockroach/commits/8165e3974c10e88b6ae11c6255872ea16f3a67e3):


```
    rsg_test.go:822: 1m15s of 5m0s: 209224 executions, 11839 successful
    rsg_test.go:822: 1m20s of 5m0s: 223943 executions, 12607 successful
    rsg_test.go:822: 1m25s of 5m0s: 238852 executions, 13318 successful
    rsg_test.go:822: 1m30s of 5m0s: 253679 executions, 14038 successful
    rsg_test.go:822: 1m35s of 5m0s: 267551 executions, 14734 successful
    rsg_test.go:822: 1m40s of 5m0s: 281268 executions, 15449 successful
    rsg_test.go:822: 1m45s of 5m0s: 295586 executions, 16185 successful
    rsg_test.go:822: 1m50s of 5m0s: 310174 executions, 16902 successful
    rsg_test.go:822: 1m55s of 5m0s: 324591 executions, 17632 successful
    rsg_test.go:822: 2m0s of 5m0s: 339818 executions, 18345 successful
    rsg_test.go:822: 2m5s of 5m0s: 354649 executions, 19044 successful
    rsg_test.go:822: 2m10s of 5m0s: 369034 executions, 19748 successful
    rsg_test.go:822: 2m15s of 5m0s: 383167 executions, 20462 successful
    rsg_test.go:822: 2m20s of 5m0s: 397292 executions, 21107 successful
    rsg_test.go:822: 2m25s of 5m0s: 411923 executions, 21707 successful
    rsg_test.go:822: 2m30s of 5m0s: 426249 executions, 22413 successful
    rsg_test.go:822: 2m35s of 5m0s: 440462 executions, 23049 successful
    rsg_test.go:822: 2m40s of 5m0s: 455464 executions, 23696 successful
    rsg_test.go:822: 2m45s of 5m0s: 470049 executions, 24356 successful
    rsg_test.go:822: 2m50s of 5m0s: 484760 executions, 25030 successful
    rsg_test.go:822: 2m55s of 5m0s: 498969 executions, 25724 successful
    rsg_test.go:822: 3m0s of 5m0s: 513985 executions, 26443 successful
    rsg_test.go:822: 3m5s of 5m0s: 529092 executions, 27159 successful
    rsg_test.go:822: 3m10s of 5m0s: 543747 executions, 27869 successful
    rsg_test.go:822: 3m15s of 5m0s: 558772 executions, 28559 successful
    rsg_test.go:822: 3m20s of 5m0s: 574074 executions, 29255 successful
    rsg_test.go:822: 3m25s of 5m0s: 589825 executions, 29936 successful
    rsg_test.go:822: 3m30s of 5m0s: 604922 executions, 30574 successful
    rsg_test.go:822: 3m35s of 5m0s: 618966 executions, 31227 successful
    rsg_test.go:822: 3m40s of 5m0s: 632573 executions, 31840 successful
    rsg_test.go:822: 3m45s of 5m0s: 647450 executions, 32486 successful
    rsg_test.go:822: 3m50s of 5m0s: 662059 executions, 33184 successful
    rsg_test.go:822: 3m55s of 5m0s: 676359 executions, 33887 successful
    rsg_test.go:822: 4m0s of 5m0s: 691185 executions, 34559 successful
    rsg_test.go:822: 4m5s of 5m0s: 705939 executions, 35221 successful
    rsg_test.go:822: 4m10s of 5m0s: 721297 executions, 35921 successful
    rsg_test.go:822: 4m15s of 5m0s: 736592 executions, 36631 successful
    rsg_test.go:822: 4m20s of 5m0s: 752153 executions, 37309 successful
    rsg_test.go:822: 4m25s of 5m0s: 767275 executions, 38024 successful
    rsg_test.go:822: 4m30s of 5m0s: 782288 executions, 38736 successful
    rsg_test.go:822: 4m35s of 5m0s: 797172 executions, 39464 successful
    rsg_test.go:822: 4m40s of 5m0s: 811571 executions, 40125 successful
    rsg_test.go:822: 4m45s of 5m0s: 825706 executions, 40793 successful
    rsg_test.go:822: 4m50s of 5m0s: 840172 executions, 41418 successful
    rsg_test.go:822: 4m55s of 5m0s: 854292 executions, 42073 successful
    rsg_test.go:858: 868792 executions, 42777 successful
    rsg_test.go:866: cannot parse output of Format: sql="PREPARE OLD_KMS AS BACKUP INTO ( THROTTLING , PLACEHOLDER ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="PREPARE old_kms AS BACKUP INTO ('throttling', 'placeholder') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2964042847
--- FAIL: TestRandomSyntaxGeneration (301.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7879910?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7879910?buildTab=artifacts#/) on master @ [ec095bc2fdbe4e518b076db20e4920fab67222bf](https://github.com/cockroachdb/cockroach/commits/ec095bc2fdbe4e518b076db20e4920fab67222bf):


```
    rsg_test.go:822: 1m20s of 5m0s: 221584 executions, 12279 successful
    rsg_test.go:822: 1m25s of 5m0s: 235862 executions, 12993 successful
    rsg_test.go:822: 1m30s of 5m0s: 250395 executions, 13757 successful
    rsg_test.go:822: 1m35s of 5m0s: 265230 executions, 14474 successful
    rsg_test.go:822: 1m40s of 5m0s: 279726 executions, 15159 successful
    rsg_test.go:822: 1m45s of 5m0s: 293350 executions, 15829 successful
    rsg_test.go:822: 1m50s of 5m0s: 307819 executions, 16561 successful
    rsg_test.go:822: 1m55s of 5m0s: 322650 executions, 17272 successful
    rsg_test.go:822: 2m0s of 5m0s: 336703 executions, 17933 successful
    rsg_test.go:822: 2m5s of 5m0s: 351839 executions, 18682 successful
    rsg_test.go:822: 2m10s of 5m0s: 366200 executions, 19364 successful
    rsg_test.go:822: 2m15s of 5m0s: 380252 executions, 20041 successful
    rsg_test.go:822: 2m20s of 5m0s: 393634 executions, 20702 successful
    rsg_test.go:822: 2m25s of 5m0s: 407411 executions, 21356 successful
    rsg_test.go:822: 2m30s of 5m0s: 421482 executions, 22003 successful
    rsg_test.go:822: 2m35s of 5m0s: 435582 executions, 22660 successful
    rsg_test.go:822: 2m40s of 5m0s: 449597 executions, 23289 successful
    rsg_test.go:822: 2m45s of 5m0s: 463350 executions, 23948 successful
    rsg_test.go:822: 2m50s of 5m0s: 477639 executions, 24588 successful
    rsg_test.go:822: 2m55s of 5m0s: 492256 executions, 25293 successful
    rsg_test.go:822: 3m0s of 5m0s: 506019 executions, 25931 successful
    rsg_test.go:822: 3m5s of 5m0s: 520408 executions, 26637 successful
    rsg_test.go:822: 3m10s of 5m0s: 534986 executions, 27303 successful
    rsg_test.go:822: 3m15s of 5m0s: 549411 executions, 27974 successful
    rsg_test.go:822: 3m20s of 5m0s: 564051 executions, 28642 successful
    rsg_test.go:822: 3m25s of 5m0s: 578246 executions, 29317 successful
    rsg_test.go:822: 3m30s of 5m0s: 593026 executions, 30043 successful
    rsg_test.go:822: 3m35s of 5m0s: 608024 executions, 30731 successful
    rsg_test.go:822: 3m40s of 5m0s: 622817 executions, 31465 successful
    rsg_test.go:822: 3m45s of 5m0s: 638033 executions, 32166 successful
    rsg_test.go:822: 3m50s of 5m0s: 651810 executions, 32841 successful
    rsg_test.go:822: 3m55s of 5m0s: 665442 executions, 33523 successful
    rsg_test.go:822: 4m0s of 5m0s: 679447 executions, 34212 successful
    rsg_test.go:822: 4m5s of 5m0s: 692966 executions, 34858 successful
    rsg_test.go:822: 4m10s of 5m0s: 707135 executions, 35531 successful
    rsg_test.go:822: 4m15s of 5m0s: 721092 executions, 36154 successful
    rsg_test.go:822: 4m20s of 5m0s: 734195 executions, 36776 successful
    rsg_test.go:822: 4m25s of 5m0s: 747946 executions, 37404 successful
    rsg_test.go:822: 4m30s of 5m0s: 762752 executions, 38077 successful
    rsg_test.go:822: 4m35s of 5m0s: 776987 executions, 38787 successful
    rsg_test.go:822: 4m40s of 5m0s: 791799 executions, 39450 successful
    rsg_test.go:822: 4m45s of 5m0s: 806525 executions, 40072 successful
    rsg_test.go:822: 4m50s of 5m0s: 820378 executions, 40700 successful
    rsg_test.go:822: 4m55s of 5m0s: 834616 executions, 41401 successful
    rsg_test.go:822: 5m0s of 5m0s: 848217 executions, 42073 successful
    rsg_test.go:858: 848218 executions, 42074 successful
    rsg_test.go:866: cannot parse output of Format: sql="ALTER BACKUP SCHEDULE 451 SET WITH DETACHED = FALSE , SET FULL BACKUP ALWAYS", formattedSQL="ALTER BACKUP SCHEDULE 451 SET WITH , SET FULL BACKUP ALWAYS": at or near ",": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2143310519
--- FAIL: TestRandomSyntaxGeneration (301.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7895032?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7895032?buildTab=artifacts#/) on master @ [24854994805cede37e6845ee2a94e10272b5506b](https://github.com/cockroachdb/cockroach/commits/24854994805cede37e6845ee2a94e10272b5506b):


```
    rsg_test.go:822: 1m20s of 5m0s: 219290 executions, 11966 successful
    rsg_test.go:822: 1m25s of 5m0s: 232987 executions, 12663 successful
    rsg_test.go:822: 1m30s of 5m0s: 247526 executions, 13366 successful
    rsg_test.go:822: 1m35s of 5m0s: 261662 executions, 14043 successful
    rsg_test.go:822: 1m40s of 5m0s: 275786 executions, 14684 successful
    rsg_test.go:822: 1m45s of 5m0s: 289617 executions, 15335 successful
    rsg_test.go:822: 1m50s of 5m0s: 303717 executions, 16040 successful
    rsg_test.go:822: 1m55s of 5m0s: 318361 executions, 16748 successful
    rsg_test.go:822: 2m0s of 5m0s: 332937 executions, 17429 successful
    rsg_test.go:822: 2m5s of 5m0s: 347512 executions, 18099 successful
    rsg_test.go:822: 2m10s of 5m0s: 361928 executions, 18761 successful
    rsg_test.go:822: 2m15s of 5m0s: 376788 executions, 19465 successful
    rsg_test.go:822: 2m20s of 5m0s: 391833 executions, 20215 successful
    rsg_test.go:822: 2m25s of 5m0s: 406730 executions, 20913 successful
    rsg_test.go:822: 2m30s of 5m0s: 421322 executions, 21586 successful
    rsg_test.go:822: 2m35s of 5m0s: 435305 executions, 22238 successful
    rsg_test.go:822: 2m40s of 5m0s: 450142 executions, 22923 successful
    rsg_test.go:822: 2m45s of 5m0s: 464809 executions, 23611 successful
    rsg_test.go:822: 2m50s of 5m0s: 479848 executions, 24282 successful
    rsg_test.go:822: 2m55s of 5m0s: 494086 executions, 24976 successful
    rsg_test.go:822: 3m0s of 5m0s: 508571 executions, 25650 successful
    rsg_test.go:822: 3m5s of 5m0s: 523377 executions, 26347 successful
    rsg_test.go:822: 3m10s of 5m0s: 537681 executions, 27046 successful
    rsg_test.go:822: 3m15s of 5m0s: 552394 executions, 27708 successful
    rsg_test.go:822: 3m20s of 5m0s: 567163 executions, 28376 successful
    rsg_test.go:822: 3m25s of 5m0s: 581229 executions, 29007 successful
    rsg_test.go:822: 3m30s of 5m0s: 594686 executions, 29660 successful
    rsg_test.go:822: 3m35s of 5m0s: 608309 executions, 30303 successful
    rsg_test.go:822: 3m40s of 5m0s: 622581 executions, 31023 successful
    rsg_test.go:822: 3m45s of 5m0s: 637656 executions, 31692 successful
    rsg_test.go:822: 3m50s of 5m0s: 652624 executions, 32439 successful
    rsg_test.go:822: 3m55s of 5m0s: 668133 executions, 33167 successful
    rsg_test.go:822: 4m0s of 5m0s: 682465 executions, 33842 successful
    rsg_test.go:822: 4m5s of 5m0s: 696940 executions, 34479 successful
    rsg_test.go:822: 4m10s of 5m0s: 711772 executions, 35178 successful
    rsg_test.go:822: 4m15s of 5m0s: 726492 executions, 35905 successful
    rsg_test.go:822: 4m20s of 5m0s: 741479 executions, 36555 successful
    rsg_test.go:822: 4m25s of 5m0s: 756299 executions, 37299 successful
    rsg_test.go:822: 4m30s of 5m0s: 771528 executions, 37982 successful
    rsg_test.go:822: 4m35s of 5m0s: 786165 executions, 38672 successful
    rsg_test.go:822: 4m40s of 5m0s: 800618 executions, 39333 successful
    rsg_test.go:822: 4m45s of 5m0s: 814957 executions, 39995 successful
    rsg_test.go:822: 4m50s of 5m0s: 829137 executions, 40644 successful
    rsg_test.go:822: 4m55s of 5m0s: 842932 executions, 41315 successful
    rsg_test.go:822: 5m0s of 5m0s: 856996 executions, 41961 successful
    rsg_test.go:858: 856997 executions, 41961 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO ( 'string' ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3314424378
--- FAIL: TestRandomSyntaxGeneration (301.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7909428?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7909428?buildTab=artifacts#/) on master @ [f2b00e8039af6ea8887ec124dad8daf19da6fbf1](https://github.com/cockroachdb/cockroach/commits/f2b00e8039af6ea8887ec124dad8daf19da6fbf1):


```
    rsg_test.go:822: 1m15s of 5m0s: 208302 executions, 11711 successful
    rsg_test.go:822: 1m20s of 5m0s: 222398 executions, 12438 successful
    rsg_test.go:822: 1m25s of 5m0s: 236825 executions, 13188 successful
    rsg_test.go:822: 1m30s of 5m0s: 251544 executions, 13927 successful
    rsg_test.go:822: 1m35s of 5m0s: 266483 executions, 14650 successful
    rsg_test.go:822: 1m40s of 5m0s: 281526 executions, 15406 successful
    rsg_test.go:822: 1m45s of 5m0s: 296300 executions, 16141 successful
    rsg_test.go:822: 1m50s of 5m0s: 310795 executions, 16809 successful
    rsg_test.go:822: 1m55s of 5m0s: 325041 executions, 17548 successful
    rsg_test.go:822: 2m0s of 5m0s: 339537 executions, 18206 successful
    rsg_test.go:822: 2m5s of 5m0s: 353552 executions, 18858 successful
    rsg_test.go:822: 2m10s of 5m0s: 367019 executions, 19525 successful
    rsg_test.go:822: 2m15s of 5m0s: 381067 executions, 20257 successful
    rsg_test.go:822: 2m20s of 5m0s: 395587 executions, 20951 successful
    rsg_test.go:822: 2m25s of 5m0s: 409748 executions, 21612 successful
    rsg_test.go:822: 2m30s of 5m0s: 424216 executions, 22298 successful
    rsg_test.go:822: 2m35s of 5m0s: 438444 executions, 22938 successful
    rsg_test.go:822: 2m40s of 5m0s: 453849 executions, 23595 successful
    rsg_test.go:822: 2m45s of 5m0s: 468720 executions, 24302 successful
    rsg_test.go:822: 2m50s of 5m0s: 483613 executions, 25002 successful
    rsg_test.go:822: 2m55s of 5m0s: 498242 executions, 25676 successful
    rsg_test.go:822: 3m0s of 5m0s: 512901 executions, 26424 successful
    rsg_test.go:822: 3m5s of 5m0s: 527829 executions, 27114 successful
    rsg_test.go:822: 3m10s of 5m0s: 541805 executions, 27791 successful
    rsg_test.go:822: 3m15s of 5m0s: 556763 executions, 28461 successful
    rsg_test.go:822: 3m20s of 5m0s: 571795 executions, 29137 successful
    rsg_test.go:822: 3m25s of 5m0s: 586933 executions, 29816 successful
    rsg_test.go:822: 3m30s of 5m0s: 602539 executions, 30557 successful
    rsg_test.go:822: 3m35s of 5m0s: 617585 executions, 31308 successful
    rsg_test.go:822: 3m40s of 5m0s: 631887 executions, 31988 successful
    rsg_test.go:822: 3m45s of 5m0s: 646989 executions, 32725 successful
    rsg_test.go:822: 3m50s of 5m0s: 662059 executions, 33431 successful
    rsg_test.go:822: 3m55s of 5m0s: 677137 executions, 34176 successful
    rsg_test.go:822: 4m0s of 5m0s: 691541 executions, 34807 successful
    rsg_test.go:822: 4m5s of 5m0s: 706341 executions, 35432 successful
    rsg_test.go:822: 4m10s of 5m0s: 720464 executions, 36095 successful
    rsg_test.go:822: 4m15s of 5m0s: 734483 executions, 36798 successful
    rsg_test.go:822: 4m20s of 5m0s: 748052 executions, 37424 successful
    rsg_test.go:822: 4m25s of 5m0s: 762348 executions, 38144 successful
    rsg_test.go:822: 4m30s of 5m0s: 776987 executions, 38811 successful
    rsg_test.go:822: 4m35s of 5m0s: 791956 executions, 39496 successful
    rsg_test.go:822: 4m40s of 5m0s: 807211 executions, 40163 successful
    rsg_test.go:822: 4m45s of 5m0s: 822134 executions, 40831 successful
    rsg_test.go:822: 4m50s of 5m0s: 837294 executions, 41579 successful
    rsg_test.go:822: 4m55s of 5m0s: 851820 executions, 42299 successful
    rsg_test.go:858: 865114 executions, 42908 successful
    rsg_test.go:866: cannot parse output of Format: sql="CREATE SCHEDULE FOR BACKUP INTO ILIKE WITH DETACHED = FALSE RECURRING 'string' FULL BACKUP ALWAYS WITH SCHEDULE OPTIONS ( 'string' = 'string' , NEXT )", formattedSQL="CREATE SCHEDULE FOR BACKUP INTO 'ilike' WITH  RECURRING 'string' FULL BACKUP ALWAYS WITH SCHEDULE OPTIONS \"string\" = 'string', next": at or near "recurring": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2703344780
--- FAIL: TestRandomSyntaxGeneration (300.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7916367?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7916367?buildTab=artifacts#/) on master @ [c050c9b4b57ecf2ceb5d449c31c617fe12c920e0](https://github.com/cockroachdb/cockroach/commits/c050c9b4b57ecf2ceb5d449c31c617fe12c920e0):


```
    rsg_test.go:822: 1m20s of 5m0s: 211191 executions, 11792 successful
    rsg_test.go:822: 1m25s of 5m0s: 224234 executions, 12466 successful
    rsg_test.go:822: 1m30s of 5m0s: 237642 executions, 13128 successful
    rsg_test.go:822: 1m35s of 5m0s: 251503 executions, 13776 successful
    rsg_test.go:822: 1m40s of 5m0s: 265704 executions, 14461 successful
    rsg_test.go:822: 1m45s of 5m0s: 279736 executions, 15150 successful
    rsg_test.go:822: 1m50s of 5m0s: 293246 executions, 15781 successful
    rsg_test.go:822: 1m55s of 5m0s: 307207 executions, 16479 successful
    rsg_test.go:822: 2m0s of 5m0s: 321316 executions, 17184 successful
    rsg_test.go:822: 2m5s of 5m0s: 335438 executions, 17840 successful
    rsg_test.go:822: 2m10s of 5m0s: 349423 executions, 18479 successful
    rsg_test.go:822: 2m15s of 5m0s: 363309 executions, 19099 successful
    rsg_test.go:822: 2m20s of 5m0s: 376637 executions, 19772 successful
    rsg_test.go:822: 2m25s of 5m0s: 390903 executions, 20474 successful
    rsg_test.go:822: 2m30s of 5m0s: 405339 executions, 21142 successful
    rsg_test.go:822: 2m35s of 5m0s: 419204 executions, 21810 successful
    rsg_test.go:822: 2m40s of 5m0s: 433117 executions, 22457 successful
    rsg_test.go:822: 2m45s of 5m0s: 447025 executions, 23118 successful
    rsg_test.go:822: 2m50s of 5m0s: 460625 executions, 23780 successful
    rsg_test.go:822: 2m55s of 5m0s: 474797 executions, 24474 successful
    rsg_test.go:822: 3m0s of 5m0s: 488735 executions, 25122 successful
    rsg_test.go:822: 3m5s of 5m0s: 502997 executions, 25782 successful
    rsg_test.go:822: 3m10s of 5m0s: 517365 executions, 26452 successful
    rsg_test.go:822: 3m15s of 5m0s: 531339 executions, 27121 successful
    rsg_test.go:822: 3m20s of 5m0s: 545566 executions, 27773 successful
    rsg_test.go:822: 3m25s of 5m0s: 559105 executions, 28405 successful
    rsg_test.go:822: 3m30s of 5m0s: 572318 executions, 28992 successful
    rsg_test.go:822: 3m35s of 5m0s: 586281 executions, 29652 successful
    rsg_test.go:822: 3m40s of 5m0s: 600163 executions, 30340 successful
    rsg_test.go:822: 3m45s of 5m0s: 614012 executions, 31002 successful
    rsg_test.go:822: 3m50s of 5m0s: 627267 executions, 31651 successful
    rsg_test.go:822: 3m55s of 5m0s: 640891 executions, 32285 successful
    rsg_test.go:822: 4m0s of 5m0s: 654924 executions, 32948 successful
    rsg_test.go:822: 4m5s of 5m0s: 669467 executions, 33623 successful
    rsg_test.go:822: 4m10s of 5m0s: 683415 executions, 34293 successful
    rsg_test.go:822: 4m15s of 5m0s: 697122 executions, 34921 successful
    rsg_test.go:822: 4m20s of 5m0s: 710926 executions, 35567 successful
    rsg_test.go:822: 4m25s of 5m0s: 725223 executions, 36221 successful
    rsg_test.go:822: 4m30s of 5m0s: 739239 executions, 36894 successful
    rsg_test.go:822: 4m35s of 5m0s: 752989 executions, 37526 successful
    rsg_test.go:822: 4m40s of 5m0s: 766135 executions, 38131 successful
    rsg_test.go:822: 4m45s of 5m0s: 779850 executions, 38748 successful
    rsg_test.go:822: 4m50s of 5m0s: 794362 executions, 39391 successful
    rsg_test.go:822: 4m55s of 5m0s: 808456 executions, 40048 successful
    rsg_test.go:822: 5m0s of 5m0s: 822978 executions, 40719 successful
    rsg_test.go:858: 822979 executions, 40719 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO LATEST IN ( ident ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO LATEST IN 'ident' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration478616399
--- FAIL: TestRandomSyntaxGeneration (301.49s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7923316?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7923316?buildTab=artifacts#/) on master @ [942a4d468e9c8ad0ef45a7be33f0a326dfb19fef](https://github.com/cockroachdb/cockroach/commits/942a4d468e9c8ad0ef45a7be33f0a326dfb19fef):


```
    rsg_test.go:822: 1m15s of 5m0s: 211767 executions, 11861 successful
    rsg_test.go:822: 1m20s of 5m0s: 226659 executions, 12567 successful
    rsg_test.go:822: 1m25s of 5m0s: 240835 executions, 13273 successful
    rsg_test.go:822: 1m30s of 5m0s: 254995 executions, 13967 successful
    rsg_test.go:822: 1m35s of 5m0s: 268881 executions, 14614 successful
    rsg_test.go:822: 1m40s of 5m0s: 283661 executions, 15328 successful
    rsg_test.go:822: 1m45s of 5m0s: 298131 executions, 15999 successful
    rsg_test.go:822: 1m50s of 5m0s: 312206 executions, 16659 successful
    rsg_test.go:822: 1m55s of 5m0s: 326100 executions, 17320 successful
    rsg_test.go:822: 2m0s of 5m0s: 339489 executions, 17982 successful
    rsg_test.go:822: 2m5s of 5m0s: 352623 executions, 18655 successful
    rsg_test.go:822: 2m10s of 5m0s: 366814 executions, 19341 successful
    rsg_test.go:822: 2m15s of 5m0s: 381403 executions, 20044 successful
    rsg_test.go:822: 2m20s of 5m0s: 395485 executions, 20701 successful
    rsg_test.go:822: 2m25s of 5m0s: 409243 executions, 21356 successful
    rsg_test.go:822: 2m30s of 5m0s: 423607 executions, 22036 successful
    rsg_test.go:822: 2m35s of 5m0s: 438265 executions, 22713 successful
    rsg_test.go:822: 2m40s of 5m0s: 452897 executions, 23356 successful
    rsg_test.go:822: 2m45s of 5m0s: 467566 executions, 24037 successful
    rsg_test.go:822: 2m50s of 5m0s: 482568 executions, 24743 successful
    rsg_test.go:822: 2m55s of 5m0s: 497747 executions, 25403 successful
    rsg_test.go:822: 3m0s of 5m0s: 512571 executions, 26049 successful
    rsg_test.go:822: 3m5s of 5m0s: 526821 executions, 26737 successful
    rsg_test.go:822: 3m10s of 5m0s: 541985 executions, 27484 successful
    rsg_test.go:822: 3m15s of 5m0s: 557064 executions, 28236 successful
    rsg_test.go:822: 3m20s of 5m0s: 572307 executions, 28937 successful
    rsg_test.go:822: 3m25s of 5m0s: 587122 executions, 29652 successful
    rsg_test.go:822: 3m30s of 5m0s: 601536 executions, 30364 successful
    rsg_test.go:822: 3m35s of 5m0s: 615877 executions, 31030 successful
    rsg_test.go:822: 3m40s of 5m0s: 630784 executions, 31680 successful
    rsg_test.go:822: 3m45s of 5m0s: 644954 executions, 32341 successful
    rsg_test.go:822: 3m50s of 5m0s: 659406 executions, 33040 successful
    rsg_test.go:822: 3m55s of 5m0s: 674083 executions, 33755 successful
    rsg_test.go:822: 4m0s of 5m0s: 689005 executions, 34481 successful
    rsg_test.go:822: 4m5s of 5m0s: 703474 executions, 35155 successful
    rsg_test.go:822: 4m10s of 5m0s: 718035 executions, 35845 successful
    rsg_test.go:822: 4m15s of 5m0s: 733109 executions, 36496 successful
    rsg_test.go:822: 4m20s of 5m0s: 748425 executions, 37192 successful
    rsg_test.go:822: 4m25s of 5m0s: 763538 executions, 37910 successful
    rsg_test.go:822: 4m30s of 5m0s: 778563 executions, 38643 successful
    rsg_test.go:822: 4m35s of 5m0s: 793807 executions, 39376 successful
    rsg_test.go:822: 4m40s of 5m0s: 808470 executions, 40033 successful
    rsg_test.go:822: 4m45s of 5m0s: 822666 executions, 40645 successful
    rsg_test.go:822: 4m50s of 5m0s: 837584 executions, 41306 successful
    rsg_test.go:822: 4m55s of 5m0s: 851968 executions, 41972 successful
    rsg_test.go:858: 865033 executions, 42607 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP TENANT 243 INTO ( 'string' ) AS OF SYSTEM TIME b'bytes' WITH DETACHED = FALSE", formattedSQL="BACKUP TENANT 243 INTO 'string' AS OF SYSTEM TIME b'bytes' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration4219410594
--- FAIL: TestRandomSyntaxGeneration (301.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7937047?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7937047?buildTab=artifacts#/) on master @ [a80652b2e4691ea76ea49e797b1b9e0998e1d61f](https://github.com/cockroachdb/cockroach/commits/a80652b2e4691ea76ea49e797b1b9e0998e1d61f):


```
    rsg_test.go:822: 1m15s of 5m0s: 191929 executions, 10466 successful
    rsg_test.go:822: 1m20s of 5m0s: 204760 executions, 11166 successful
    rsg_test.go:822: 1m25s of 5m0s: 218755 executions, 11864 successful
    rsg_test.go:822: 1m30s of 5m0s: 234433 executions, 12614 successful
    rsg_test.go:822: 1m35s of 5m0s: 249887 executions, 13349 successful
    rsg_test.go:822: 1m40s of 5m0s: 265354 executions, 14179 successful
    rsg_test.go:822: 1m45s of 5m0s: 281175 executions, 14983 successful
    rsg_test.go:822: 1m50s of 5m0s: 296793 executions, 15759 successful
    rsg_test.go:822: 1m55s of 5m0s: 311673 executions, 16491 successful
    rsg_test.go:822: 2m0s of 5m0s: 326344 executions, 17216 successful
    rsg_test.go:822: 2m5s of 5m0s: 341228 executions, 17903 successful
    rsg_test.go:822: 2m10s of 5m0s: 355917 executions, 18542 successful
    rsg_test.go:822: 2m15s of 5m0s: 369576 executions, 19119 successful
    rsg_test.go:822: 2m20s of 5m0s: 383104 executions, 19696 successful
    rsg_test.go:822: 2m25s of 5m0s: 395745 executions, 20293 successful
    rsg_test.go:822: 2m30s of 5m0s: 408644 executions, 20815 successful
    rsg_test.go:822: 2m35s of 5m0s: 420687 executions, 21318 successful
    rsg_test.go:822: 2m40s of 5m0s: 432524 executions, 21846 successful
    rsg_test.go:822: 2m45s of 5m0s: 444339 executions, 22377 successful
    rsg_test.go:822: 2m50s of 5m0s: 456730 executions, 22870 successful
    rsg_test.go:822: 2m55s of 5m0s: 469310 executions, 23383 successful
    rsg_test.go:822: 3m0s of 5m0s: 480883 executions, 23858 successful
    rsg_test.go:822: 3m5s of 5m0s: 492053 executions, 24336 successful
    rsg_test.go:822: 3m10s of 5m0s: 503882 executions, 24811 successful
    rsg_test.go:822: 3m15s of 5m0s: 514790 executions, 25275 successful
    rsg_test.go:822: 3m20s of 5m0s: 525604 executions, 25760 successful
    rsg_test.go:822: 3m25s of 5m0s: 536291 executions, 26236 successful
    rsg_test.go:822: 3m30s of 5m0s: 548036 executions, 26662 successful
    rsg_test.go:822: 3m35s of 5m0s: 560137 executions, 27157 successful
    rsg_test.go:822: 3m40s of 5m0s: 572249 executions, 27648 successful
    rsg_test.go:822: 3m45s of 5m0s: 583800 executions, 28129 successful
    rsg_test.go:822: 3m50s of 5m0s: 595374 executions, 28565 successful
    rsg_test.go:822: 3m55s of 5m0s: 607548 executions, 29025 successful
    rsg_test.go:822: 4m0s of 5m0s: 618545 executions, 29443 successful
    rsg_test.go:822: 4m5s of 5m0s: 629635 executions, 29879 successful
    rsg_test.go:822: 4m10s of 5m0s: 641420 executions, 30379 successful
    rsg_test.go:822: 4m15s of 5m0s: 652615 executions, 30881 successful
    rsg_test.go:822: 4m20s of 5m0s: 664552 executions, 31336 successful
    rsg_test.go:822: 4m25s of 5m0s: 675884 executions, 31873 successful
    rsg_test.go:822: 4m30s of 5m0s: 687569 executions, 32370 successful
    rsg_test.go:822: 4m35s of 5m0s: 698413 executions, 32797 successful
    rsg_test.go:822: 4m40s of 5m0s: 707749 executions, 33161 successful
    rsg_test.go:822: 4m45s of 5m0s: 716608 executions, 33528 successful
    rsg_test.go:822: 4m50s of 5m0s: 726400 executions, 33963 successful
    rsg_test.go:822: 4m55s of 5m0s: 736530 executions, 34405 successful
    rsg_test.go:858: 745868 executions, 34796 successful
    rsg_test.go:866: cannot parse output of Format: sql="PREPARE POINT ( ident ARRAY ) AS BACKUP TO ( 'string' , ASENSITIVE ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="PREPARE \"point\" (ident[]) AS BACKUP TO ('string', 'asensitive') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration112144599
--- FAIL: TestRandomSyntaxGeneration (301.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7952899?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7952899?buildTab=artifacts#/) on master @ [052acc88ad9d7296ce6b8b441627fb469cc74d95](https://github.com/cockroachdb/cockroach/commits/052acc88ad9d7296ce6b8b441627fb469cc74d95):


```
    rsg_test.go:822: 1m20s of 5m0s: 247708 executions, 13544 successful
    rsg_test.go:822: 1m25s of 5m0s: 264386 executions, 14402 successful
    rsg_test.go:822: 1m30s of 5m0s: 280795 executions, 15192 successful
    rsg_test.go:822: 1m35s of 5m0s: 297044 executions, 15882 successful
    rsg_test.go:822: 1m40s of 5m0s: 313035 executions, 16538 successful
    rsg_test.go:822: 1m45s of 5m0s: 328985 executions, 17147 successful
    rsg_test.go:822: 1m50s of 5m0s: 343399 executions, 17760 successful
    rsg_test.go:822: 1m55s of 5m0s: 358602 executions, 18385 successful
    rsg_test.go:822: 2m0s of 5m0s: 374013 executions, 19015 successful
    rsg_test.go:822: 2m5s of 5m0s: 389274 executions, 19642 successful
    rsg_test.go:822: 2m10s of 5m0s: 404331 executions, 20353 successful
    rsg_test.go:822: 2m15s of 5m0s: 420012 executions, 21087 successful
    rsg_test.go:822: 2m20s of 5m0s: 436003 executions, 21838 successful
    rsg_test.go:822: 2m25s of 5m0s: 450740 executions, 22547 successful
    rsg_test.go:822: 2m30s of 5m0s: 465594 executions, 23297 successful
    rsg_test.go:822: 2m35s of 5m0s: 480410 executions, 24009 successful
    rsg_test.go:822: 2m40s of 5m0s: 495220 executions, 24651 successful
    rsg_test.go:822: 2m45s of 5m0s: 509585 executions, 25278 successful
    rsg_test.go:822: 2m50s of 5m0s: 523039 executions, 25876 successful
    rsg_test.go:822: 2m55s of 5m0s: 536996 executions, 26453 successful
    rsg_test.go:822: 3m0s of 5m0s: 553820 executions, 27233 successful
    rsg_test.go:822: 3m5s of 5m0s: 570661 executions, 27987 successful
    rsg_test.go:822: 3m10s of 5m0s: 587434 executions, 28818 successful
    rsg_test.go:822: 3m15s of 5m0s: 603766 executions, 29581 successful
    rsg_test.go:822: 3m20s of 5m0s: 620605 executions, 30344 successful
    rsg_test.go:822: 3m25s of 5m0s: 636839 executions, 31020 successful
    rsg_test.go:822: 3m30s of 5m0s: 652501 executions, 31624 successful
    rsg_test.go:822: 3m35s of 5m0s: 668068 executions, 32221 successful
    rsg_test.go:822: 3m40s of 5m0s: 683661 executions, 32914 successful
    rsg_test.go:822: 3m45s of 5m0s: 700405 executions, 33669 successful
    rsg_test.go:822: 3m50s of 5m0s: 717241 executions, 34443 successful
    rsg_test.go:822: 3m55s of 5m0s: 733158 executions, 35075 successful
    rsg_test.go:822: 4m0s of 5m0s: 749386 executions, 35708 successful
    rsg_test.go:822: 4m5s of 5m0s: 765042 executions, 36393 successful
    rsg_test.go:822: 4m10s of 5m0s: 780653 executions, 37027 successful
    rsg_test.go:822: 4m15s of 5m0s: 795577 executions, 37660 successful
    rsg_test.go:822: 4m20s of 5m0s: 809707 executions, 38236 successful
    rsg_test.go:822: 4m25s of 5m0s: 823707 executions, 38754 successful
    rsg_test.go:822: 4m30s of 5m0s: 837302 executions, 39345 successful
    rsg_test.go:822: 4m35s of 5m0s: 849191 executions, 39859 successful
    rsg_test.go:822: 4m40s of 5m0s: 861800 executions, 40392 successful
    rsg_test.go:822: 4m45s of 5m0s: 873269 executions, 40847 successful
    rsg_test.go:822: 4m50s of 5m0s: 885756 executions, 41366 successful
    rsg_test.go:822: 4m55s of 5m0s: 897614 executions, 41848 successful
    rsg_test.go:822: 5m0s of 5m0s: 909828 executions, 42353 successful
    rsg_test.go:858: 909829 executions, 42353 successful
    rsg_test.go:866: cannot parse output of Format: sql="PREPARE DISCARD AS BACKUP INTO 'string' IN ( 'string' ) WITH DETACHED = FALSE", formattedSQL="PREPARE discard AS BACKUP INTO 'string' IN 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3724464652
--- FAIL: TestRandomSyntaxGeneration (301.44s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7969269?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7969269?buildTab=artifacts#/) on master @ [d6f98e90684894fd36f53596e6aac355676d232e](https://github.com/cockroachdb/cockroach/commits/d6f98e90684894fd36f53596e6aac355676d232e):


```
    rsg_test.go:822: 1m15s of 5m0s: 209557 executions, 11581 successful
    rsg_test.go:822: 1m20s of 5m0s: 223961 executions, 12297 successful
    rsg_test.go:822: 1m25s of 5m0s: 237863 executions, 12980 successful
    rsg_test.go:822: 1m30s of 5m0s: 252329 executions, 13666 successful
    rsg_test.go:822: 1m35s of 5m0s: 266385 executions, 14345 successful
    rsg_test.go:822: 1m40s of 5m0s: 281156 executions, 15060 successful
    rsg_test.go:822: 1m45s of 5m0s: 295557 executions, 15812 successful
    rsg_test.go:822: 1m50s of 5m0s: 310888 executions, 16540 successful
    rsg_test.go:822: 1m55s of 5m0s: 325372 executions, 17215 successful
    rsg_test.go:822: 2m0s of 5m0s: 339997 executions, 17922 successful
    rsg_test.go:822: 2m5s of 5m0s: 354216 executions, 18619 successful
    rsg_test.go:822: 2m10s of 5m0s: 368278 executions, 19268 successful
    rsg_test.go:822: 2m15s of 5m0s: 382670 executions, 19994 successful
    rsg_test.go:822: 2m20s of 5m0s: 397392 executions, 20678 successful
    rsg_test.go:822: 2m25s of 5m0s: 411457 executions, 21308 successful
    rsg_test.go:822: 2m30s of 5m0s: 426800 executions, 22007 successful
    rsg_test.go:822: 2m35s of 5m0s: 441789 executions, 22675 successful
    rsg_test.go:822: 2m40s of 5m0s: 455913 executions, 23344 successful
    rsg_test.go:822: 2m45s of 5m0s: 470540 executions, 24016 successful
    rsg_test.go:822: 2m50s of 5m0s: 484915 executions, 24657 successful
    rsg_test.go:822: 2m55s of 5m0s: 498769 executions, 25305 successful
    rsg_test.go:822: 3m0s of 5m0s: 512387 executions, 25937 successful
    rsg_test.go:822: 3m5s of 5m0s: 526566 executions, 26633 successful
    rsg_test.go:822: 3m10s of 5m0s: 539658 executions, 27285 successful
    rsg_test.go:822: 3m15s of 5m0s: 552940 executions, 27949 successful
    rsg_test.go:822: 3m20s of 5m0s: 567165 executions, 28622 successful
    rsg_test.go:822: 3m25s of 5m0s: 581279 executions, 29277 successful
    rsg_test.go:822: 3m30s of 5m0s: 595391 executions, 29966 successful
    rsg_test.go:822: 3m35s of 5m0s: 610299 executions, 30663 successful
    rsg_test.go:822: 3m40s of 5m0s: 625154 executions, 31362 successful
    rsg_test.go:822: 3m45s of 5m0s: 639753 executions, 32014 successful
    rsg_test.go:822: 3m50s of 5m0s: 654613 executions, 32724 successful
    rsg_test.go:822: 3m55s of 5m0s: 669165 executions, 33417 successful
    rsg_test.go:822: 4m0s of 5m0s: 683655 executions, 34134 successful
    rsg_test.go:822: 4m5s of 5m0s: 698391 executions, 34845 successful
    rsg_test.go:822: 4m10s of 5m0s: 712913 executions, 35460 successful
    rsg_test.go:822: 4m15s of 5m0s: 727205 executions, 36151 successful
    rsg_test.go:822: 4m20s of 5m0s: 742317 executions, 36840 successful
    rsg_test.go:822: 4m25s of 5m0s: 758149 executions, 37613 successful
    rsg_test.go:822: 4m30s of 5m0s: 773268 executions, 38261 successful
    rsg_test.go:822: 4m35s of 5m0s: 788885 executions, 38961 successful
    rsg_test.go:822: 4m40s of 5m0s: 803624 executions, 39663 successful
    rsg_test.go:822: 4m45s of 5m0s: 818083 executions, 40286 successful
    rsg_test.go:822: 4m50s of 5m0s: 831900 executions, 40940 successful
    rsg_test.go:822: 4m55s of 5m0s: 846274 executions, 41598 successful
    rsg_test.go:858: 859716 executions, 42263 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP TO ( PLACEHOLDER , PLACEHOLDER , PLACEHOLDER , PLACEHOLDER ) INCREMENTAL FROM RUNNING WITH DETACHED = FALSE", formattedSQL="BACKUP TO ('placeholder', 'placeholder', 'placeholder', 'placeholder') INCREMENTAL FROM 'running' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3602222174
--- FAIL: TestRandomSyntaxGeneration (301.37s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7985126?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7985126?buildTab=artifacts#/) on master @ [dcdf599e9eaee8010137167f419509ccb7627406](https://github.com/cockroachdb/cockroach/commits/dcdf599e9eaee8010137167f419509ccb7627406):


```
    rsg_test.go:822: 1m15s of 5m0s: 211328 executions, 11882 successful
    rsg_test.go:822: 1m20s of 5m0s: 225280 executions, 12576 successful
    rsg_test.go:822: 1m25s of 5m0s: 239563 executions, 13272 successful
    rsg_test.go:822: 1m30s of 5m0s: 253758 executions, 14005 successful
    rsg_test.go:822: 1m35s of 5m0s: 268392 executions, 14684 successful
    rsg_test.go:822: 1m40s of 5m0s: 283477 executions, 15427 successful
    rsg_test.go:822: 1m45s of 5m0s: 298733 executions, 16097 successful
    rsg_test.go:822: 1m50s of 5m0s: 313884 executions, 16754 successful
    rsg_test.go:822: 1m55s of 5m0s: 328151 executions, 17468 successful
    rsg_test.go:822: 2m0s of 5m0s: 342171 executions, 18150 successful
    rsg_test.go:822: 2m5s of 5m0s: 355890 executions, 18745 successful
    rsg_test.go:822: 2m10s of 5m0s: 369707 executions, 19286 successful
    rsg_test.go:822: 2m15s of 5m0s: 383240 executions, 19858 successful
    rsg_test.go:822: 2m20s of 5m0s: 396737 executions, 20446 successful
    rsg_test.go:822: 2m25s of 5m0s: 410472 executions, 21035 successful
    rsg_test.go:822: 2m30s of 5m0s: 424137 executions, 21584 successful
    rsg_test.go:822: 2m35s of 5m0s: 436514 executions, 22048 successful
    rsg_test.go:822: 2m40s of 5m0s: 449537 executions, 22533 successful
    rsg_test.go:822: 2m45s of 5m0s: 461846 executions, 23065 successful
    rsg_test.go:822: 2m50s of 5m0s: 474346 executions, 23573 successful
    rsg_test.go:822: 2m55s of 5m0s: 486483 executions, 24075 successful
    rsg_test.go:822: 3m0s of 5m0s: 498606 executions, 24624 successful
    rsg_test.go:822: 3m5s of 5m0s: 514477 executions, 25340 successful
    rsg_test.go:822: 3m10s of 5m0s: 529833 executions, 26074 successful
    rsg_test.go:822: 3m15s of 5m0s: 544273 executions, 26725 successful
    rsg_test.go:822: 3m20s of 5m0s: 558608 executions, 27407 successful
    rsg_test.go:822: 3m25s of 5m0s: 573312 executions, 28067 successful
    rsg_test.go:822: 3m30s of 5m0s: 588474 executions, 28768 successful
    rsg_test.go:822: 3m35s of 5m0s: 603323 executions, 29480 successful
    rsg_test.go:822: 3m40s of 5m0s: 617467 executions, 30133 successful
    rsg_test.go:822: 3m45s of 5m0s: 631315 executions, 30782 successful
    rsg_test.go:822: 3m50s of 5m0s: 645314 executions, 31418 successful
    rsg_test.go:822: 3m55s of 5m0s: 659536 executions, 32099 successful
    rsg_test.go:822: 4m0s of 5m0s: 673367 executions, 32755 successful
    rsg_test.go:822: 4m5s of 5m0s: 687426 executions, 33428 successful
    rsg_test.go:822: 4m10s of 5m0s: 701756 executions, 34146 successful
    rsg_test.go:822: 4m15s of 5m0s: 716039 executions, 34821 successful
    rsg_test.go:822: 4m20s of 5m0s: 730407 executions, 35484 successful
    rsg_test.go:822: 4m25s of 5m0s: 744977 executions, 36175 successful
    rsg_test.go:822: 4m30s of 5m0s: 759584 executions, 36864 successful
    rsg_test.go:822: 4m35s of 5m0s: 774155 executions, 37519 successful
    rsg_test.go:822: 4m40s of 5m0s: 788656 executions, 38195 successful
    rsg_test.go:822: 4m45s of 5m0s: 803834 executions, 38890 successful
    rsg_test.go:822: 4m50s of 5m0s: 818460 executions, 39574 successful
    rsg_test.go:822: 4m55s of 5m0s: 832560 executions, 40171 successful
    rsg_test.go:858: 847114 executions, 40801 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO 'string' WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2337204885
--- FAIL: TestRandomSyntaxGeneration (301.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8000031?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8000031?buildTab=artifacts#/) on master @ [7c8e7012b73a598f3aa1da9bbf422e833706f5d3](https://github.com/cockroachdb/cockroach/commits/7c8e7012b73a598f3aa1da9bbf422e833706f5d3):


```
    rsg_test.go:822: 1m20s of 5m0s: 215537 executions, 11877 successful
    rsg_test.go:822: 1m25s of 5m0s: 228898 executions, 12530 successful
    rsg_test.go:822: 1m30s of 5m0s: 242269 executions, 13191 successful
    rsg_test.go:822: 1m35s of 5m0s: 255225 executions, 13827 successful
    rsg_test.go:822: 1m40s of 5m0s: 269052 executions, 14537 successful
    rsg_test.go:822: 1m45s of 5m0s: 283677 executions, 15239 successful
    rsg_test.go:822: 1m50s of 5m0s: 298031 executions, 15945 successful
    rsg_test.go:822: 1m55s of 5m0s: 312159 executions, 16647 successful
    rsg_test.go:822: 2m0s of 5m0s: 325597 executions, 17289 successful
    rsg_test.go:822: 2m5s of 5m0s: 339783 executions, 17973 successful
    rsg_test.go:822: 2m10s of 5m0s: 354407 executions, 18667 successful
    rsg_test.go:822: 2m15s of 5m0s: 368424 executions, 19318 successful
    rsg_test.go:822: 2m20s of 5m0s: 382448 executions, 19999 successful
    rsg_test.go:822: 2m25s of 5m0s: 396373 executions, 20612 successful
    rsg_test.go:822: 2m30s of 5m0s: 410803 executions, 21243 successful
    rsg_test.go:822: 2m35s of 5m0s: 425326 executions, 21842 successful
    rsg_test.go:822: 2m40s of 5m0s: 439080 executions, 22388 successful
    rsg_test.go:822: 2m45s of 5m0s: 453423 executions, 22975 successful
    rsg_test.go:822: 2m50s of 5m0s: 467824 executions, 23557 successful
    rsg_test.go:822: 2m55s of 5m0s: 482605 executions, 24139 successful
    rsg_test.go:822: 3m0s of 5m0s: 496775 executions, 24718 successful
    rsg_test.go:822: 3m5s of 5m0s: 509849 executions, 25284 successful
    rsg_test.go:822: 3m10s of 5m0s: 524299 executions, 25874 successful
    rsg_test.go:822: 3m15s of 5m0s: 538096 executions, 26469 successful
    rsg_test.go:822: 3m20s of 5m0s: 551843 executions, 27064 successful
    rsg_test.go:822: 3m25s of 5m0s: 566111 executions, 27630 successful
    rsg_test.go:822: 3m30s of 5m0s: 580250 executions, 28195 successful
    rsg_test.go:822: 3m35s of 5m0s: 593632 executions, 28808 successful
    rsg_test.go:822: 3m40s of 5m0s: 607640 executions, 29382 successful
    rsg_test.go:822: 3m45s of 5m0s: 621359 executions, 29931 successful
    rsg_test.go:822: 3m50s of 5m0s: 635644 executions, 30538 successful
    rsg_test.go:822: 3m55s of 5m0s: 649103 executions, 31106 successful
    rsg_test.go:822: 4m0s of 5m0s: 662558 executions, 31679 successful
    rsg_test.go:822: 4m5s of 5m0s: 675405 executions, 32225 successful
    rsg_test.go:822: 4m10s of 5m0s: 687676 executions, 32775 successful
    rsg_test.go:822: 4m15s of 5m0s: 700147 executions, 33249 successful
    rsg_test.go:822: 4m20s of 5m0s: 712724 executions, 33756 successful
    rsg_test.go:822: 4m25s of 5m0s: 725311 executions, 34228 successful
    rsg_test.go:822: 4m30s of 5m0s: 736856 executions, 34688 successful
    rsg_test.go:822: 4m35s of 5m0s: 747791 executions, 35155 successful
    rsg_test.go:822: 4m40s of 5m0s: 758815 executions, 35662 successful
    rsg_test.go:822: 4m45s of 5m0s: 769504 executions, 36096 successful
    rsg_test.go:822: 4m50s of 5m0s: 779359 executions, 36494 successful
    rsg_test.go:822: 4m55s of 5m0s: 790357 executions, 36967 successful
    rsg_test.go:822: 5m0s of 5m0s: 801136 executions, 37455 successful
    rsg_test.go:858: 801137 executions, 37456 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP TO ( PLACEHOLDER ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TO 'placeholder' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration1863156745
--- FAIL: TestRandomSyntaxGeneration (301.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8014306?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8014306?buildTab=artifacts#/) on master @ [94e03c016955dfd64d4e15358ea92226a8f362aa](https://github.com/cockroachdb/cockroach/commits/94e03c016955dfd64d4e15358ea92226a8f362aa):


```
    rsg_test.go:822: 1m20s of 5m0s: 211371 executions, 10700 successful
    rsg_test.go:822: 1m25s of 5m0s: 224617 executions, 11393 successful
    rsg_test.go:822: 1m30s of 5m0s: 238363 executions, 12045 successful
    rsg_test.go:822: 1m35s of 5m0s: 251353 executions, 12601 successful
    rsg_test.go:822: 1m40s of 5m0s: 264518 executions, 13106 successful
    rsg_test.go:822: 1m45s of 5m0s: 278046 executions, 13707 successful
    rsg_test.go:822: 1m50s of 5m0s: 290910 executions, 14234 successful
    rsg_test.go:822: 1m55s of 5m0s: 303388 executions, 14745 successful
    rsg_test.go:822: 2m0s of 5m0s: 315887 executions, 15302 successful
    rsg_test.go:822: 2m5s of 5m0s: 328644 executions, 15786 successful
    rsg_test.go:822: 2m10s of 5m0s: 341421 executions, 16279 successful
    rsg_test.go:822: 2m15s of 5m0s: 353244 executions, 16778 successful
    rsg_test.go:822: 2m20s of 5m0s: 364898 executions, 17262 successful
    rsg_test.go:822: 2m25s of 5m0s: 376393 executions, 17746 successful
    rsg_test.go:822: 2m30s of 5m0s: 388617 executions, 18280 successful
    rsg_test.go:822: 2m35s of 5m0s: 400250 executions, 18752 successful
    rsg_test.go:822: 2m40s of 5m0s: 412058 executions, 19232 successful
    rsg_test.go:822: 2m45s of 5m0s: 423646 executions, 19716 successful
    rsg_test.go:822: 2m50s of 5m0s: 434261 executions, 20169 successful
    rsg_test.go:822: 2m55s of 5m0s: 445066 executions, 20620 successful
    rsg_test.go:822: 3m0s of 5m0s: 455799 executions, 21097 successful
    rsg_test.go:822: 3m5s of 5m0s: 466131 executions, 21502 successful
    rsg_test.go:822: 3m10s of 5m0s: 476795 executions, 21948 successful
    rsg_test.go:822: 3m15s of 5m0s: 486677 executions, 22359 successful
    rsg_test.go:822: 3m20s of 5m0s: 496128 executions, 22775 successful
    rsg_test.go:822: 3m25s of 5m0s: 506925 executions, 23239 successful
    rsg_test.go:822: 3m30s of 5m0s: 517558 executions, 23715 successful
    rsg_test.go:822: 3m35s of 5m0s: 527902 executions, 24171 successful
    rsg_test.go:822: 3m40s of 5m0s: 538535 executions, 24616 successful
    rsg_test.go:822: 3m45s of 5m0s: 548505 executions, 25033 successful
    rsg_test.go:822: 3m50s of 5m0s: 558841 executions, 25461 successful
    rsg_test.go:822: 3m55s of 5m0s: 569110 executions, 25889 successful
    rsg_test.go:822: 4m0s of 5m0s: 579537 executions, 26364 successful
    rsg_test.go:822: 4m5s of 5m0s: 590135 executions, 26784 successful
    rsg_test.go:822: 4m10s of 5m0s: 600863 executions, 27229 successful
    rsg_test.go:822: 4m15s of 5m0s: 611558 executions, 27700 successful
    rsg_test.go:822: 4m20s of 5m0s: 621701 executions, 28088 successful
    rsg_test.go:822: 4m25s of 5m0s: 632143 executions, 28572 successful
    rsg_test.go:822: 4m30s of 5m0s: 642749 executions, 29026 successful
    rsg_test.go:822: 4m35s of 5m0s: 653161 executions, 29483 successful
    rsg_test.go:822: 4m40s of 5m0s: 663121 executions, 29901 successful
    rsg_test.go:822: 4m45s of 5m0s: 673162 executions, 30303 successful
    rsg_test.go:822: 4m50s of 5m0s: 683886 executions, 30788 successful
    rsg_test.go:822: 4m55s of 5m0s: 697622 executions, 31386 successful
    rsg_test.go:822: 5m0s of 5m0s: 711830 executions, 32064 successful
    rsg_test.go:858: 711831 executions, 32065 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP TO ( NATURAL ) INCREMENTAL FROM 'string' , FAMILY , PLACEHOLDER , 'string' WITH DETACHED = FALSE", formattedSQL="BACKUP TO 'natural' INCREMENTAL FROM 'string', 'family', 'placeholder', 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration891425456
--- FAIL: TestRandomSyntaxGeneration (301.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8030993?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8030993?buildTab=artifacts#/) on master @ [61f7c816574848f804f1adf7f42f7315bfcaaeba](https://github.com/cockroachdb/cockroach/commits/61f7c816574848f804f1adf7f42f7315bfcaaeba):


```
    rsg_test.go:822: 1m20s of 5m0s: 216747 executions, 10888 successful
    rsg_test.go:822: 1m25s of 5m0s: 230377 executions, 11519 successful
    rsg_test.go:822: 1m30s of 5m0s: 244588 executions, 12269 successful
    rsg_test.go:822: 1m35s of 5m0s: 258277 executions, 12896 successful
    rsg_test.go:822: 1m40s of 5m0s: 271777 executions, 13553 successful
    rsg_test.go:822: 1m45s of 5m0s: 284167 executions, 14196 successful
    rsg_test.go:822: 1m50s of 5m0s: 296979 executions, 14771 successful
    rsg_test.go:822: 1m55s of 5m0s: 308863 executions, 15357 successful
    rsg_test.go:822: 2m0s of 5m0s: 321974 executions, 15952 successful
    rsg_test.go:822: 2m5s of 5m0s: 335570 executions, 16580 successful
    rsg_test.go:822: 2m10s of 5m0s: 349310 executions, 17170 successful
    rsg_test.go:822: 2m15s of 5m0s: 362282 executions, 17816 successful
    rsg_test.go:822: 2m20s of 5m0s: 376324 executions, 18478 successful
    rsg_test.go:822: 2m25s of 5m0s: 389458 executions, 19123 successful
    rsg_test.go:822: 2m30s of 5m0s: 403197 executions, 19744 successful
    rsg_test.go:822: 2m35s of 5m0s: 417127 executions, 20443 successful
    rsg_test.go:822: 2m40s of 5m0s: 430943 executions, 21094 successful
    rsg_test.go:822: 2m45s of 5m0s: 445208 executions, 21761 successful
    rsg_test.go:822: 2m50s of 5m0s: 459244 executions, 22492 successful
    rsg_test.go:822: 2m55s of 5m0s: 473027 executions, 23180 successful
    rsg_test.go:822: 3m0s of 5m0s: 486712 executions, 23848 successful
    rsg_test.go:822: 3m5s of 5m0s: 500168 executions, 24542 successful
    rsg_test.go:822: 3m10s of 5m0s: 513661 executions, 25222 successful
    rsg_test.go:822: 3m15s of 5m0s: 527106 executions, 25889 successful
    rsg_test.go:822: 3m20s of 5m0s: 541470 executions, 26541 successful
    rsg_test.go:822: 3m25s of 5m0s: 556097 executions, 27205 successful
    rsg_test.go:822: 3m30s of 5m0s: 570418 executions, 27892 successful
    rsg_test.go:822: 3m35s of 5m0s: 584379 executions, 28617 successful
    rsg_test.go:822: 3m40s of 5m0s: 598349 executions, 29306 successful
    rsg_test.go:822: 3m45s of 5m0s: 611888 executions, 29928 successful
    rsg_test.go:822: 3m50s of 5m0s: 625624 executions, 30587 successful
    rsg_test.go:822: 3m55s of 5m0s: 639321 executions, 31256 successful
    rsg_test.go:822: 4m0s of 5m0s: 653270 executions, 31856 successful
    rsg_test.go:822: 4m5s of 5m0s: 667265 executions, 32524 successful
    rsg_test.go:822: 4m10s of 5m0s: 680867 executions, 33204 successful
    rsg_test.go:822: 4m15s of 5m0s: 695440 executions, 33909 successful
    rsg_test.go:822: 4m20s of 5m0s: 709386 executions, 34567 successful
    rsg_test.go:822: 4m25s of 5m0s: 722953 executions, 35211 successful
    rsg_test.go:822: 4m30s of 5m0s: 735936 executions, 35818 successful
    rsg_test.go:822: 4m35s of 5m0s: 748444 executions, 36391 successful
    rsg_test.go:822: 4m40s of 5m0s: 761100 executions, 36977 successful
    rsg_test.go:822: 4m45s of 5m0s: 773809 executions, 37572 successful
    rsg_test.go:822: 4m50s of 5m0s: 786420 executions, 38127 successful
    rsg_test.go:822: 4m55s of 5m0s: 798913 executions, 38615 successful
    rsg_test.go:822: 5m0s of 5m0s: 811673 executions, 39169 successful
    rsg_test.go:858: 811676 executions, 39171 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INDEX_AFTER_ORDER_BY_BEFORE_AT . GEOMETRYZ . ident INTO 'string' IN 'string' WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TABLE \"index_after_order_by_before_at\".geometryz.ident INTO 'string' IN 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration1370825382
--- FAIL: TestRandomSyntaxGeneration (301.49s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8046103?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8046103?buildTab=artifacts#/) on master @ [10266a323f94c3cf397d5de590e512d987e63e22](https://github.com/cockroachdb/cockroach/commits/10266a323f94c3cf397d5de590e512d987e63e22):


```
    rsg_test.go:822: 1m15s of 5m0s: 206923 executions, 11165 successful
    rsg_test.go:822: 1m20s of 5m0s: 220158 executions, 11813 successful
    rsg_test.go:822: 1m25s of 5m0s: 233761 executions, 12456 successful
    rsg_test.go:822: 1m30s of 5m0s: 248080 executions, 13228 successful
    rsg_test.go:822: 1m35s of 5m0s: 263100 executions, 13996 successful
    rsg_test.go:822: 1m40s of 5m0s: 277759 executions, 14744 successful
    rsg_test.go:822: 1m45s of 5m0s: 292785 executions, 15530 successful
    rsg_test.go:822: 1m50s of 5m0s: 307561 executions, 16220 successful
    rsg_test.go:822: 1m55s of 5m0s: 322051 executions, 16910 successful
    rsg_test.go:822: 2m0s of 5m0s: 336312 executions, 17617 successful
    rsg_test.go:822: 2m5s of 5m0s: 350783 executions, 18300 successful
    rsg_test.go:822: 2m10s of 5m0s: 365768 executions, 18975 successful
    rsg_test.go:822: 2m15s of 5m0s: 379667 executions, 19560 successful
    rsg_test.go:822: 2m20s of 5m0s: 393970 executions, 20146 successful
    rsg_test.go:822: 2m25s of 5m0s: 408372 executions, 20693 successful
    rsg_test.go:822: 2m30s of 5m0s: 422091 executions, 21234 successful
    rsg_test.go:822: 2m35s of 5m0s: 435134 executions, 21832 successful
    rsg_test.go:822: 2m40s of 5m0s: 449407 executions, 22496 successful
    rsg_test.go:822: 2m45s of 5m0s: 463491 executions, 23135 successful
    rsg_test.go:822: 2m50s of 5m0s: 477262 executions, 23794 successful
    rsg_test.go:822: 2m55s of 5m0s: 491856 executions, 24461 successful
    rsg_test.go:822: 3m0s of 5m0s: 506450 executions, 25044 successful
    rsg_test.go:822: 3m5s of 5m0s: 521108 executions, 25623 successful
    rsg_test.go:822: 3m10s of 5m0s: 535150 executions, 26163 successful
    rsg_test.go:822: 3m15s of 5m0s: 549438 executions, 26770 successful
    rsg_test.go:822: 3m20s of 5m0s: 563535 executions, 27394 successful
    rsg_test.go:822: 3m25s of 5m0s: 577440 executions, 27984 successful
    rsg_test.go:822: 3m30s of 5m0s: 591244 executions, 28527 successful
    rsg_test.go:822: 3m35s of 5m0s: 604334 executions, 29078 successful
    rsg_test.go:822: 3m40s of 5m0s: 617111 executions, 29606 successful
    rsg_test.go:822: 3m45s of 5m0s: 629780 executions, 30152 successful
    rsg_test.go:822: 3m50s of 5m0s: 643573 executions, 30733 successful
    rsg_test.go:822: 3m55s of 5m0s: 658521 executions, 31456 successful
    rsg_test.go:822: 4m0s of 5m0s: 673711 executions, 32200 successful
    rsg_test.go:822: 4m5s of 5m0s: 688927 executions, 32935 successful
    rsg_test.go:822: 4m10s of 5m0s: 702797 executions, 33637 successful
    rsg_test.go:822: 4m15s of 5m0s: 717636 executions, 34352 successful
    rsg_test.go:822: 4m20s of 5m0s: 731931 executions, 35019 successful
    rsg_test.go:822: 4m25s of 5m0s: 746352 executions, 35711 successful
    rsg_test.go:822: 4m30s of 5m0s: 761224 executions, 36356 successful
    rsg_test.go:822: 4m35s of 5m0s: 775927 executions, 36934 successful
    rsg_test.go:822: 4m40s of 5m0s: 790113 executions, 37531 successful
    rsg_test.go:822: 4m45s of 5m0s: 804004 executions, 38149 successful
    rsg_test.go:822: 4m50s of 5m0s: 817479 executions, 38780 successful
    rsg_test.go:822: 4m55s of 5m0s: 831797 executions, 39428 successful
    rsg_test.go:858: 845506 executions, 40054 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO ( PLACEHOLDER , PLACEHOLDER , PLACEHOLDER , PLACEHOLDER , PLACEHOLDER ) WITH DETACHED = FALSE", formattedSQL="BACKUP INTO ('placeholder', 'placeholder', 'placeholder', 'placeholder', 'placeholder') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2758229244
--- FAIL: TestRandomSyntaxGeneration (301.44s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8061583?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8061583?buildTab=artifacts#/) on master @ [15bc0c4780681d95e705c22dbe2e7d8d190054ef](https://github.com/cockroachdb/cockroach/commits/15bc0c4780681d95e705c22dbe2e7d8d190054ef):


```
    rsg_test.go:822: 1m20s of 5m0s: 218167 executions, 12006 successful
    rsg_test.go:822: 1m25s of 5m0s: 232413 executions, 12732 successful
    rsg_test.go:822: 1m30s of 5m0s: 246180 executions, 13392 successful
    rsg_test.go:822: 1m35s of 5m0s: 260559 executions, 14080 successful
    rsg_test.go:822: 1m40s of 5m0s: 274889 executions, 14685 successful
    rsg_test.go:822: 1m45s of 5m0s: 288702 executions, 15275 successful
    rsg_test.go:822: 1m50s of 5m0s: 301540 executions, 15801 successful
    rsg_test.go:822: 1m55s of 5m0s: 315802 executions, 16430 successful
    rsg_test.go:822: 2m0s of 5m0s: 329659 executions, 17026 successful
    rsg_test.go:822: 2m5s of 5m0s: 343170 executions, 17585 successful
    rsg_test.go:822: 2m10s of 5m0s: 356150 executions, 18105 successful
    rsg_test.go:822: 2m15s of 5m0s: 369065 executions, 18671 successful
    rsg_test.go:822: 2m20s of 5m0s: 382337 executions, 19199 successful
    rsg_test.go:822: 2m25s of 5m0s: 393942 executions, 19675 successful
    rsg_test.go:822: 2m30s of 5m0s: 406918 executions, 20157 successful
    rsg_test.go:822: 2m35s of 5m0s: 419410 executions, 20640 successful
    rsg_test.go:822: 2m40s of 5m0s: 430449 executions, 21106 successful
    rsg_test.go:822: 2m45s of 5m0s: 441910 executions, 21562 successful
    rsg_test.go:822: 2m50s of 5m0s: 453245 executions, 22002 successful
    rsg_test.go:822: 2m55s of 5m0s: 464865 executions, 22479 successful
    rsg_test.go:822: 3m0s of 5m0s: 476527 executions, 22930 successful
    rsg_test.go:822: 3m5s of 5m0s: 487874 executions, 23324 successful
    rsg_test.go:822: 3m10s of 5m0s: 499586 executions, 23784 successful
    rsg_test.go:822: 3m15s of 5m0s: 510339 executions, 24161 successful
    rsg_test.go:822: 3m20s of 5m0s: 521472 executions, 24670 successful
    rsg_test.go:822: 3m25s of 5m0s: 532611 executions, 25140 successful
    rsg_test.go:822: 3m30s of 5m0s: 543686 executions, 25633 successful
    rsg_test.go:822: 3m35s of 5m0s: 553299 executions, 26052 successful
    rsg_test.go:822: 3m40s of 5m0s: 564428 executions, 26530 successful
    rsg_test.go:822: 3m45s of 5m0s: 576448 executions, 26965 successful
    rsg_test.go:822: 3m50s of 5m0s: 587726 executions, 27439 successful
    rsg_test.go:822: 3m55s of 5m0s: 598659 executions, 27927 successful
    rsg_test.go:822: 4m0s of 5m0s: 609700 executions, 28390 successful
    rsg_test.go:822: 4m5s of 5m0s: 620731 executions, 28856 successful
    rsg_test.go:822: 4m10s of 5m0s: 631420 executions, 29297 successful
    rsg_test.go:822: 4m15s of 5m0s: 640519 executions, 29691 successful
    rsg_test.go:822: 4m20s of 5m0s: 651123 executions, 30128 successful
    rsg_test.go:822: 4m25s of 5m0s: 661316 executions, 30518 successful
    rsg_test.go:822: 4m30s of 5m0s: 670723 executions, 30929 successful
    rsg_test.go:822: 4m35s of 5m0s: 679886 executions, 31313 successful
    rsg_test.go:822: 4m40s of 5m0s: 689095 executions, 31709 successful
    rsg_test.go:822: 4m45s of 5m0s: 698684 executions, 32104 successful
    rsg_test.go:822: 4m50s of 5m0s: 708725 executions, 32472 successful
    rsg_test.go:822: 4m55s of 5m0s: 718446 executions, 32876 successful
    rsg_test.go:822: 5m0s of 5m0s: 728174 executions, 33243 successful
    rsg_test.go:858: 728176 executions, 33243 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO QUERY WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO 'query' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration4254875383
--- FAIL: TestRandomSyntaxGeneration (301.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8078821?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8078821?buildTab=artifacts#/) on master @ [4232883add85a151c423c45904ac4096d04656c5](https://github.com/cockroachdb/cockroach/commits/4232883add85a151c423c45904ac4096d04656c5):


```
    rsg_test.go:822: 1m15s of 5m0s: 191662 executions, 10908 successful
    rsg_test.go:822: 1m20s of 5m0s: 205432 executions, 11573 successful
    rsg_test.go:822: 1m25s of 5m0s: 219257 executions, 12221 successful
    rsg_test.go:822: 1m30s of 5m0s: 232085 executions, 12800 successful
    rsg_test.go:822: 1m35s of 5m0s: 244480 executions, 13430 successful
    rsg_test.go:822: 1m40s of 5m0s: 257774 executions, 14081 successful
    rsg_test.go:822: 1m45s of 5m0s: 270930 executions, 14716 successful
    rsg_test.go:822: 1m50s of 5m0s: 284576 executions, 15396 successful
    rsg_test.go:822: 1m55s of 5m0s: 298184 executions, 16055 successful
    rsg_test.go:822: 2m0s of 5m0s: 310540 executions, 16700 successful
    rsg_test.go:822: 2m5s of 5m0s: 322551 executions, 17275 successful
    rsg_test.go:822: 2m10s of 5m0s: 335264 executions, 17886 successful
    rsg_test.go:822: 2m15s of 5m0s: 348399 executions, 18502 successful
    rsg_test.go:822: 2m20s of 5m0s: 360335 executions, 18985 successful
    rsg_test.go:822: 2m25s of 5m0s: 372878 executions, 19456 successful
    rsg_test.go:822: 2m30s of 5m0s: 385297 executions, 19943 successful
    rsg_test.go:822: 2m35s of 5m0s: 397188 executions, 20466 successful
    rsg_test.go:822: 2m40s of 5m0s: 409344 executions, 20994 successful
    rsg_test.go:822: 2m45s of 5m0s: 421303 executions, 21485 successful
    rsg_test.go:822: 2m50s of 5m0s: 433015 executions, 21991 successful
    rsg_test.go:822: 2m55s of 5m0s: 444796 executions, 22443 successful
    rsg_test.go:822: 3m0s of 5m0s: 457037 executions, 22926 successful
    rsg_test.go:822: 3m5s of 5m0s: 469078 executions, 23421 successful
    rsg_test.go:822: 3m10s of 5m0s: 481777 executions, 23983 successful
    rsg_test.go:822: 3m15s of 5m0s: 495387 executions, 24632 successful
    rsg_test.go:822: 3m20s of 5m0s: 509025 executions, 25284 successful
    rsg_test.go:822: 3m25s of 5m0s: 523071 executions, 25895 successful
    rsg_test.go:822: 3m30s of 5m0s: 536208 executions, 26431 successful
    rsg_test.go:822: 3m35s of 5m0s: 548813 executions, 26936 successful
    rsg_test.go:822: 3m40s of 5m0s: 561953 executions, 27494 successful
    rsg_test.go:822: 3m45s of 5m0s: 575078 executions, 28063 successful
    rsg_test.go:822: 3m50s of 5m0s: 588127 executions, 28691 successful
    rsg_test.go:822: 3m55s of 5m0s: 601328 executions, 29311 successful
    rsg_test.go:822: 4m0s of 5m0s: 614764 executions, 29947 successful
    rsg_test.go:822: 4m5s of 5m0s: 628232 executions, 30558 successful
    rsg_test.go:822: 4m10s of 5m0s: 641994 executions, 31261 successful
    rsg_test.go:822: 4m15s of 5m0s: 655678 executions, 31938 successful
    rsg_test.go:822: 4m20s of 5m0s: 669714 executions, 32568 successful
    rsg_test.go:822: 4m25s of 5m0s: 683253 executions, 33187 successful
    rsg_test.go:822: 4m30s of 5m0s: 697292 executions, 33871 successful
    rsg_test.go:822: 4m35s of 5m0s: 711405 executions, 34458 successful
    rsg_test.go:822: 4m40s of 5m0s: 725751 executions, 35138 successful
    rsg_test.go:822: 4m45s of 5m0s: 739615 executions, 35791 successful
    rsg_test.go:822: 4m50s of 5m0s: 753763 executions, 36448 successful
    rsg_test.go:822: 4m55s of 5m0s: 767798 executions, 37112 successful
    rsg_test.go:858: 781671 executions, 37761 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO LATEST IN ( PLACEHOLDER ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO LATEST IN 'placeholder' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration4089065762
--- FAIL: TestRandomSyntaxGeneration (300.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8087939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8087939?buildTab=artifacts#/) on master @ [c4bde8b72cdd4016845ae70ef5162b3f11fab1fb](https://github.com/cockroachdb/cockroach/commits/c4bde8b72cdd4016845ae70ef5162b3f11fab1fb):


```
    rsg_test.go:822: 1m20s of 5m0s: 229818 executions, 12695 successful
    rsg_test.go:822: 1m25s of 5m0s: 244549 executions, 13433 successful
    rsg_test.go:822: 1m30s of 5m0s: 258734 executions, 14172 successful
    rsg_test.go:822: 1m35s of 5m0s: 273546 executions, 14938 successful
    rsg_test.go:822: 1m40s of 5m0s: 288150 executions, 15676 successful
    rsg_test.go:822: 1m45s of 5m0s: 302678 executions, 16448 successful
    rsg_test.go:822: 1m50s of 5m0s: 317241 executions, 17175 successful
    rsg_test.go:822: 1m55s of 5m0s: 331493 executions, 17897 successful
    rsg_test.go:822: 2m0s of 5m0s: 346284 executions, 18599 successful
    rsg_test.go:822: 2m5s of 5m0s: 360320 executions, 19294 successful
    rsg_test.go:822: 2m10s of 5m0s: 373684 executions, 19969 successful
    rsg_test.go:822: 2m15s of 5m0s: 388344 executions, 20660 successful
    rsg_test.go:822: 2m20s of 5m0s: 402807 executions, 21370 successful
    rsg_test.go:822: 2m25s of 5m0s: 416468 executions, 22051 successful
    rsg_test.go:822: 2m30s of 5m0s: 429659 executions, 22690 successful
    rsg_test.go:822: 2m35s of 5m0s: 444093 executions, 23348 successful
    rsg_test.go:822: 2m40s of 5m0s: 458482 executions, 24064 successful
    rsg_test.go:822: 2m45s of 5m0s: 472557 executions, 24708 successful
    rsg_test.go:822: 2m50s of 5m0s: 487105 executions, 25369 successful
    rsg_test.go:822: 2m55s of 5m0s: 501473 executions, 25978 successful
    rsg_test.go:822: 3m0s of 5m0s: 515678 executions, 26572 successful
    rsg_test.go:822: 3m5s of 5m0s: 530354 executions, 27119 successful
    rsg_test.go:822: 3m10s of 5m0s: 544554 executions, 27723 successful
    rsg_test.go:822: 3m15s of 5m0s: 559278 executions, 28441 successful
    rsg_test.go:822: 3m20s of 5m0s: 572569 executions, 29049 successful
    rsg_test.go:822: 3m25s of 5m0s: 587398 executions, 29615 successful
    rsg_test.go:822: 3m30s of 5m0s: 601184 executions, 30179 successful
    rsg_test.go:822: 3m35s of 5m0s: 614482 executions, 30747 successful
    rsg_test.go:822: 3m40s of 5m0s: 627671 executions, 31281 successful
    rsg_test.go:822: 3m45s of 5m0s: 641633 executions, 31865 successful
    rsg_test.go:822: 3m50s of 5m0s: 654656 executions, 32408 successful
    rsg_test.go:822: 3m55s of 5m0s: 667673 executions, 32966 successful
    rsg_test.go:822: 4m0s of 5m0s: 681359 executions, 33505 successful
    rsg_test.go:822: 4m5s of 5m0s: 693828 executions, 34086 successful
    rsg_test.go:822: 4m10s of 5m0s: 706211 executions, 34634 successful
    rsg_test.go:822: 4m15s of 5m0s: 717118 executions, 35136 successful
    rsg_test.go:822: 4m20s of 5m0s: 728693 executions, 35612 successful
    rsg_test.go:822: 4m25s of 5m0s: 740412 executions, 36115 successful
    rsg_test.go:822: 4m30s of 5m0s: 752073 executions, 36593 successful
    rsg_test.go:822: 4m35s of 5m0s: 766380 executions, 37278 successful
    rsg_test.go:822: 4m40s of 5m0s: 780655 executions, 37953 successful
    rsg_test.go:822: 4m45s of 5m0s: 794710 executions, 38624 successful
    rsg_test.go:822: 4m50s of 5m0s: 808922 executions, 39324 successful
    rsg_test.go:822: 4m55s of 5m0s: 822546 executions, 39985 successful
    rsg_test.go:822: 5m0s of 5m0s: 836997 executions, 40699 successful
    rsg_test.go:858: 836998 executions, 40699 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP TABLE INDEX_AFTER_ORDER_BY_BEFORE_AT INTO 'string' IN PLACEHOLDER WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TABLE \"index_after_order_by_before_at\" INTO 'string' IN 'placeholder' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration49107078
--- FAIL: TestRandomSyntaxGeneration (301.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8094307?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8094307?buildTab=artifacts#/) on master @ [c4bde8b72cdd4016845ae70ef5162b3f11fab1fb](https://github.com/cockroachdb/cockroach/commits/c4bde8b72cdd4016845ae70ef5162b3f11fab1fb):


```
    rsg_test.go:822: 1m15s of 5m0s: 201363 executions, 11123 successful
    rsg_test.go:822: 1m20s of 5m0s: 213830 executions, 11666 successful
    rsg_test.go:822: 1m25s of 5m0s: 226152 executions, 12164 successful
    rsg_test.go:822: 1m30s of 5m0s: 238831 executions, 12677 successful
    rsg_test.go:822: 1m35s of 5m0s: 251707 executions, 13238 successful
    rsg_test.go:822: 1m40s of 5m0s: 265111 executions, 13842 successful
    rsg_test.go:822: 1m45s of 5m0s: 278192 executions, 14395 successful
    rsg_test.go:822: 1m50s of 5m0s: 291369 executions, 14977 successful
    rsg_test.go:822: 1m55s of 5m0s: 305785 executions, 15710 successful
    rsg_test.go:822: 2m0s of 5m0s: 319962 executions, 16319 successful
    rsg_test.go:822: 2m5s of 5m0s: 333471 executions, 16950 successful
    rsg_test.go:822: 2m10s of 5m0s: 348120 executions, 17553 successful
    rsg_test.go:822: 2m15s of 5m0s: 362140 executions, 18139 successful
    rsg_test.go:822: 2m20s of 5m0s: 376578 executions, 18696 successful
    rsg_test.go:822: 2m25s of 5m0s: 390689 executions, 19305 successful
    rsg_test.go:822: 2m30s of 5m0s: 404783 executions, 19906 successful
    rsg_test.go:822: 2m35s of 5m0s: 418955 executions, 20483 successful
    rsg_test.go:822: 2m40s of 5m0s: 432568 executions, 21012 successful
    rsg_test.go:822: 2m45s of 5m0s: 446524 executions, 21615 successful
    rsg_test.go:822: 2m50s of 5m0s: 460712 executions, 22218 successful
    rsg_test.go:822: 2m55s of 5m0s: 475093 executions, 22751 successful
    rsg_test.go:822: 3m0s of 5m0s: 488728 executions, 23313 successful
    rsg_test.go:822: 3m5s of 5m0s: 503260 executions, 23943 successful
    rsg_test.go:822: 3m10s of 5m0s: 517530 executions, 24622 successful
    rsg_test.go:822: 3m15s of 5m0s: 531672 executions, 25272 successful
    rsg_test.go:822: 3m20s of 5m0s: 545998 executions, 25922 successful
    rsg_test.go:822: 3m25s of 5m0s: 560057 executions, 26506 successful
    rsg_test.go:822: 3m30s of 5m0s: 574195 executions, 27103 successful
    rsg_test.go:822: 3m35s of 5m0s: 587575 executions, 27566 successful
    rsg_test.go:822: 3m40s of 5m0s: 600868 executions, 28097 successful
    rsg_test.go:822: 3m45s of 5m0s: 613286 executions, 28610 successful
    rsg_test.go:822: 3m50s of 5m0s: 626464 executions, 29134 successful
    rsg_test.go:822: 3m55s of 5m0s: 639543 executions, 29676 successful
    rsg_test.go:822: 4m0s of 5m0s: 652227 executions, 30223 successful
    rsg_test.go:822: 4m5s of 5m0s: 664191 executions, 30741 successful
    rsg_test.go:822: 4m10s of 5m0s: 675985 executions, 31209 successful
    rsg_test.go:822: 4m15s of 5m0s: 688143 executions, 31728 successful
    rsg_test.go:822: 4m20s of 5m0s: 699942 executions, 32194 successful
    rsg_test.go:822: 4m25s of 5m0s: 711944 executions, 32672 successful
    rsg_test.go:822: 4m30s of 5m0s: 723029 executions, 33162 successful
    rsg_test.go:822: 4m35s of 5m0s: 734099 executions, 33599 successful
    rsg_test.go:822: 4m40s of 5m0s: 745525 executions, 34020 successful
    rsg_test.go:822: 4m45s of 5m0s: 756890 executions, 34477 successful
    rsg_test.go:822: 4m50s of 5m0s: 767946 executions, 34876 successful
    rsg_test.go:822: 4m55s of 5m0s: 778919 executions, 35280 successful
    rsg_test.go:858: 789122 executions, 35706 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP FAMILY . EXPERIMENTAL_REPLICA INTO ( 'string' , GROUPING ) WITH DETACHED = FALSE", formattedSQL="BACKUP TABLE \"family\".experimental_replica INTO ('string', 'grouping') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3766762903
--- FAIL: TestRandomSyntaxGeneration (301.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8100531?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8100531?buildTab=artifacts#/) on master @ [c4bde8b72cdd4016845ae70ef5162b3f11fab1fb](https://github.com/cockroachdb/cockroach/commits/c4bde8b72cdd4016845ae70ef5162b3f11fab1fb):


```
    rsg_test.go:822: 1m15s of 5m0s: 194065 executions, 10364 successful
    rsg_test.go:822: 1m20s of 5m0s: 207838 executions, 11076 successful
    rsg_test.go:822: 1m25s of 5m0s: 221998 executions, 11821 successful
    rsg_test.go:822: 1m30s of 5m0s: 235651 executions, 12451 successful
    rsg_test.go:822: 1m35s of 5m0s: 249597 executions, 13135 successful
    rsg_test.go:822: 1m40s of 5m0s: 263663 executions, 13812 successful
    rsg_test.go:822: 1m45s of 5m0s: 277628 executions, 14477 successful
    rsg_test.go:822: 1m50s of 5m0s: 291468 executions, 15166 successful
    rsg_test.go:822: 1m55s of 5m0s: 305564 executions, 15885 successful
    rsg_test.go:822: 2m0s of 5m0s: 319998 executions, 16637 successful
    rsg_test.go:822: 2m5s of 5m0s: 334277 executions, 17312 successful
    rsg_test.go:822: 2m10s of 5m0s: 348225 executions, 17973 successful
    rsg_test.go:822: 2m15s of 5m0s: 361953 executions, 18589 successful
    rsg_test.go:822: 2m20s of 5m0s: 376232 executions, 19210 successful
    rsg_test.go:822: 2m25s of 5m0s: 390551 executions, 19910 successful
    rsg_test.go:822: 2m30s of 5m0s: 404698 executions, 20585 successful
    rsg_test.go:822: 2m35s of 5m0s: 418502 executions, 21246 successful
    rsg_test.go:822: 2m40s of 5m0s: 432102 executions, 21929 successful
    rsg_test.go:822: 2m45s of 5m0s: 445380 executions, 22529 successful
    rsg_test.go:822: 2m50s of 5m0s: 459359 executions, 23161 successful
    rsg_test.go:822: 2m55s of 5m0s: 473109 executions, 23673 successful
    rsg_test.go:822: 3m0s of 5m0s: 485911 executions, 24200 successful
    rsg_test.go:822: 3m5s of 5m0s: 498178 executions, 24743 successful
    rsg_test.go:822: 3m10s of 5m0s: 511304 executions, 25291 successful
    rsg_test.go:822: 3m15s of 5m0s: 523998 executions, 25793 successful
    rsg_test.go:822: 3m20s of 5m0s: 537489 executions, 26457 successful
    rsg_test.go:822: 3m25s of 5m0s: 551522 executions, 27078 successful
    rsg_test.go:822: 3m30s of 5m0s: 564815 executions, 27722 successful
    rsg_test.go:822: 3m35s of 5m0s: 578683 executions, 28399 successful
    rsg_test.go:822: 3m40s of 5m0s: 592739 executions, 29074 successful
    rsg_test.go:822: 3m45s of 5m0s: 606505 executions, 29644 successful
    rsg_test.go:822: 3m50s of 5m0s: 620123 executions, 30204 successful
    rsg_test.go:822: 3m55s of 5m0s: 634362 executions, 30739 successful
    rsg_test.go:822: 4m0s of 5m0s: 648446 executions, 31277 successful
    rsg_test.go:822: 4m5s of 5m0s: 661787 executions, 31836 successful
    rsg_test.go:822: 4m10s of 5m0s: 674951 executions, 32355 successful
    rsg_test.go:822: 4m15s of 5m0s: 688089 executions, 32853 successful
    rsg_test.go:822: 4m20s of 5m0s: 700765 executions, 33418 successful
    rsg_test.go:822: 4m25s of 5m0s: 713294 executions, 33935 successful
    rsg_test.go:822: 4m30s of 5m0s: 725945 executions, 34456 successful
    rsg_test.go:822: 4m35s of 5m0s: 738572 executions, 34992 successful
    rsg_test.go:822: 4m40s of 5m0s: 750604 executions, 35467 successful
    rsg_test.go:822: 4m45s of 5m0s: 761815 executions, 35943 successful
    rsg_test.go:822: 4m50s of 5m0s: 773386 executions, 36444 successful
    rsg_test.go:822: 4m55s of 5m0s: 784440 executions, 36881 successful
    rsg_test.go:858: 796268 executions, 37316 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO LATEST IN ( LIKE , PLACEHOLDER , PLACEHOLDER , PLACEHOLDER , PLACEHOLDER ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO LATEST IN ('like', 'placeholder', 'placeholder', 'placeholder', 'placeholder') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3903138637
--- FAIL: TestRandomSyntaxGeneration (300.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8107009?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8107009?buildTab=artifacts#/) on master @ [9c5375f6a7375724cdbcbaa0029ed97a230d7abe](https://github.com/cockroachdb/cockroach/commits/9c5375f6a7375724cdbcbaa0029ed97a230d7abe):


```
    rsg_test.go:822: 1m15s of 5m0s: 209558 executions, 11538 successful
    rsg_test.go:822: 1m20s of 5m0s: 224145 executions, 12259 successful
    rsg_test.go:822: 1m25s of 5m0s: 238720 executions, 13010 successful
    rsg_test.go:822: 1m30s of 5m0s: 252930 executions, 13768 successful
    rsg_test.go:822: 1m35s of 5m0s: 267570 executions, 14495 successful
    rsg_test.go:822: 1m40s of 5m0s: 282090 executions, 15212 successful
    rsg_test.go:822: 1m45s of 5m0s: 296441 executions, 15922 successful
    rsg_test.go:822: 1m50s of 5m0s: 310940 executions, 16637 successful
    rsg_test.go:822: 1m55s of 5m0s: 325577 executions, 17384 successful
    rsg_test.go:822: 2m0s of 5m0s: 340096 executions, 18092 successful
    rsg_test.go:822: 2m5s of 5m0s: 354812 executions, 18799 successful
    rsg_test.go:822: 2m10s of 5m0s: 369183 executions, 19450 successful
    rsg_test.go:822: 2m15s of 5m0s: 383723 executions, 20072 successful
    rsg_test.go:822: 2m20s of 5m0s: 397605 executions, 20733 successful
    rsg_test.go:822: 2m25s of 5m0s: 411936 executions, 21397 successful
    rsg_test.go:822: 2m30s of 5m0s: 425947 executions, 22072 successful
    rsg_test.go:822: 2m35s of 5m0s: 439344 executions, 22675 successful
    rsg_test.go:822: 2m40s of 5m0s: 453119 executions, 23283 successful
    rsg_test.go:822: 2m45s of 5m0s: 467674 executions, 23956 successful
    rsg_test.go:822: 2m50s of 5m0s: 482805 executions, 24637 successful
    rsg_test.go:822: 2m55s of 5m0s: 497854 executions, 25238 successful
    rsg_test.go:822: 3m0s of 5m0s: 512628 executions, 25915 successful
    rsg_test.go:822: 3m5s of 5m0s: 527482 executions, 26626 successful
    rsg_test.go:822: 3m10s of 5m0s: 542046 executions, 27342 successful
    rsg_test.go:822: 3m15s of 5m0s: 556591 executions, 28029 successful
    rsg_test.go:822: 3m20s of 5m0s: 571565 executions, 28757 successful
    rsg_test.go:822: 3m25s of 5m0s: 585974 executions, 29384 successful
    rsg_test.go:822: 3m30s of 5m0s: 600142 executions, 30043 successful
    rsg_test.go:822: 3m35s of 5m0s: 614874 executions, 30746 successful
    rsg_test.go:822: 3m40s of 5m0s: 629029 executions, 31399 successful
    rsg_test.go:822: 3m45s of 5m0s: 642842 executions, 32006 successful
    rsg_test.go:822: 3m50s of 5m0s: 657032 executions, 32576 successful
    rsg_test.go:822: 3m55s of 5m0s: 671223 executions, 33079 successful
    rsg_test.go:822: 4m0s of 5m0s: 684291 executions, 33573 successful
    rsg_test.go:822: 4m5s of 5m0s: 697314 executions, 34065 successful
    rsg_test.go:822: 4m10s of 5m0s: 710499 executions, 34614 successful
    rsg_test.go:822: 4m15s of 5m0s: 723201 executions, 35149 successful
    rsg_test.go:822: 4m20s of 5m0s: 736057 executions, 35710 successful
    rsg_test.go:822: 4m25s of 5m0s: 749084 executions, 36264 successful
    rsg_test.go:822: 4m30s of 5m0s: 762086 executions, 36846 successful
    rsg_test.go:822: 4m35s of 5m0s: 775008 executions, 37346 successful
    rsg_test.go:822: 4m40s of 5m0s: 787784 executions, 37900 successful
    rsg_test.go:822: 4m45s of 5m0s: 800484 executions, 38409 successful
    rsg_test.go:822: 4m50s of 5m0s: 813068 executions, 38954 successful
    rsg_test.go:822: 4m55s of 5m0s: 825318 executions, 39473 successful
    rsg_test.go:858: 837581 executions, 39948 successful
    rsg_test.go:866: cannot parse output of Format: sql="PREPARE BUCKET_COUNT AS BACKUP KMS INTO 'string' WITH DETACHED = FALSE", formattedSQL="PREPARE bucket_count AS BACKUP TABLE kms INTO 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration1264734409
--- FAIL: TestRandomSyntaxGeneration (300.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8117089?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8117089?buildTab=artifacts#/) on master @ [6723e00a46aaa3ea575093bd82a02b7d6f6b131b](https://github.com/cockroachdb/cockroach/commits/6723e00a46aaa3ea575093bd82a02b7d6f6b131b):


```
    rsg_test.go:822: 1m15s of 5m0s: 200899 executions, 10515 successful
    rsg_test.go:822: 1m20s of 5m0s: 213642 executions, 11115 successful
    rsg_test.go:822: 1m25s of 5m0s: 225195 executions, 11637 successful
    rsg_test.go:822: 1m30s of 5m0s: 237368 executions, 12160 successful
    rsg_test.go:822: 1m35s of 5m0s: 249711 executions, 12686 successful
    rsg_test.go:822: 1m40s of 5m0s: 262400 executions, 13223 successful
    rsg_test.go:822: 1m45s of 5m0s: 275906 executions, 13746 successful
    rsg_test.go:822: 1m50s of 5m0s: 288798 executions, 14257 successful
    rsg_test.go:822: 1m55s of 5m0s: 301761 executions, 14767 successful
    rsg_test.go:822: 2m0s of 5m0s: 314153 executions, 15269 successful
    rsg_test.go:822: 2m5s of 5m0s: 325876 executions, 15748 successful
    rsg_test.go:822: 2m10s of 5m0s: 337184 executions, 16189 successful
    rsg_test.go:822: 2m15s of 5m0s: 349627 executions, 16747 successful
    rsg_test.go:822: 2m20s of 5m0s: 362008 executions, 17262 successful
    rsg_test.go:822: 2m25s of 5m0s: 374594 executions, 17774 successful
    rsg_test.go:822: 2m30s of 5m0s: 386925 executions, 18294 successful
    rsg_test.go:822: 2m35s of 5m0s: 399652 executions, 18814 successful
    rsg_test.go:822: 2m40s of 5m0s: 411903 executions, 19367 successful
    rsg_test.go:822: 2m45s of 5m0s: 423766 executions, 19876 successful
    rsg_test.go:822: 2m50s of 5m0s: 435793 executions, 20360 successful
    rsg_test.go:822: 2m55s of 5m0s: 448213 executions, 20853 successful
    rsg_test.go:822: 3m0s of 5m0s: 460519 executions, 21323 successful
    rsg_test.go:822: 3m5s of 5m0s: 472471 executions, 21852 successful
    rsg_test.go:822: 3m10s of 5m0s: 483769 executions, 22304 successful
    rsg_test.go:822: 3m15s of 5m0s: 495570 executions, 22808 successful
    rsg_test.go:822: 3m20s of 5m0s: 506628 executions, 23207 successful
    rsg_test.go:822: 3m25s of 5m0s: 516978 executions, 23628 successful
    rsg_test.go:822: 3m30s of 5m0s: 528012 executions, 24158 successful
    rsg_test.go:822: 3m35s of 5m0s: 540079 executions, 24660 successful
    rsg_test.go:822: 3m40s of 5m0s: 551823 executions, 25127 successful
    rsg_test.go:822: 3m45s of 5m0s: 563291 executions, 25521 successful
    rsg_test.go:822: 3m50s of 5m0s: 573771 executions, 25971 successful
    rsg_test.go:822: 3m55s of 5m0s: 583323 executions, 26376 successful
    rsg_test.go:822: 4m0s of 5m0s: 594361 executions, 26840 successful
    rsg_test.go:822: 4m5s of 5m0s: 607454 executions, 27393 successful
    rsg_test.go:822: 4m10s of 5m0s: 622190 executions, 28084 successful
    rsg_test.go:822: 4m15s of 5m0s: 638058 executions, 28758 successful
    rsg_test.go:822: 4m20s of 5m0s: 653020 executions, 29427 successful
    rsg_test.go:822: 4m25s of 5m0s: 667342 executions, 30058 successful
    rsg_test.go:822: 4m30s of 5m0s: 682051 executions, 30748 successful
    rsg_test.go:822: 4m35s of 5m0s: 697141 executions, 31449 successful
    rsg_test.go:822: 4m40s of 5m0s: 711479 executions, 32143 successful
    rsg_test.go:822: 4m45s of 5m0s: 726035 executions, 32820 successful
    rsg_test.go:822: 4m50s of 5m0s: 741040 executions, 33498 successful
    rsg_test.go:822: 4m55s of 5m0s: 755428 executions, 34139 successful
    rsg_test.go:858: 768699 executions, 34785 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP SECOND INTO ( 'string' ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TABLE second INTO 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3968780319
--- FAIL: TestRandomSyntaxGeneration (301.45s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8127829?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8127829?buildTab=artifacts#/) on master @ [64e4fc9faa4e0ab19fe5ba78f053bc2b1390cb5e](https://github.com/cockroachdb/cockroach/commits/64e4fc9faa4e0ab19fe5ba78f053bc2b1390cb5e):


```
    rsg_test.go:822: 1m15s of 5m0s: 195104 executions, 10783 successful
    rsg_test.go:822: 1m20s of 5m0s: 208851 executions, 11452 successful
    rsg_test.go:822: 1m25s of 5m0s: 222438 executions, 12161 successful
    rsg_test.go:822: 1m30s of 5m0s: 236535 executions, 12821 successful
    rsg_test.go:822: 1m35s of 5m0s: 250992 executions, 13540 successful
    rsg_test.go:822: 1m40s of 5m0s: 264553 executions, 14238 successful
    rsg_test.go:822: 1m45s of 5m0s: 279103 executions, 14949 successful
    rsg_test.go:822: 1m50s of 5m0s: 293151 executions, 15637 successful
    rsg_test.go:822: 1m55s of 5m0s: 307103 executions, 16321 successful
    rsg_test.go:822: 2m0s of 5m0s: 320688 executions, 16957 successful
    rsg_test.go:822: 2m5s of 5m0s: 334196 executions, 17609 successful
    rsg_test.go:822: 2m10s of 5m0s: 347428 executions, 18210 successful
    rsg_test.go:822: 2m15s of 5m0s: 360342 executions, 18698 successful
    rsg_test.go:822: 2m20s of 5m0s: 372952 executions, 19231 successful
    rsg_test.go:822: 2m25s of 5m0s: 385561 executions, 19747 successful
    rsg_test.go:822: 2m30s of 5m0s: 397866 executions, 20365 successful
    rsg_test.go:822: 2m35s of 5m0s: 410087 executions, 21004 successful
    rsg_test.go:822: 2m40s of 5m0s: 422733 executions, 21620 successful
    rsg_test.go:822: 2m45s of 5m0s: 435286 executions, 22214 successful
    rsg_test.go:822: 2m50s of 5m0s: 448934 executions, 22868 successful
    rsg_test.go:822: 2m55s of 5m0s: 462701 executions, 23430 successful
    rsg_test.go:822: 3m0s of 5m0s: 476375 executions, 24067 successful
    rsg_test.go:822: 3m5s of 5m0s: 489594 executions, 24718 successful
    rsg_test.go:822: 3m10s of 5m0s: 502954 executions, 25313 successful
    rsg_test.go:822: 3m15s of 5m0s: 516408 executions, 25948 successful
    rsg_test.go:822: 3m20s of 5m0s: 530123 executions, 26634 successful
    rsg_test.go:822: 3m25s of 5m0s: 544050 executions, 27192 successful
    rsg_test.go:822: 3m30s of 5m0s: 558062 executions, 27736 successful
    rsg_test.go:822: 3m35s of 5m0s: 571550 executions, 28287 successful
    rsg_test.go:822: 3m40s of 5m0s: 584181 executions, 28813 successful
    rsg_test.go:822: 3m45s of 5m0s: 597094 executions, 29334 successful
    rsg_test.go:822: 3m50s of 5m0s: 609830 executions, 29859 successful
    rsg_test.go:822: 3m55s of 5m0s: 622698 executions, 30374 successful
    rsg_test.go:822: 4m0s of 5m0s: 635449 executions, 30933 successful
    rsg_test.go:822: 4m5s of 5m0s: 646980 executions, 31430 successful
    rsg_test.go:822: 4m10s of 5m0s: 658290 executions, 31913 successful
    rsg_test.go:822: 4m15s of 5m0s: 669797 executions, 32413 successful
    rsg_test.go:822: 4m20s of 5m0s: 680528 executions, 32878 successful
    rsg_test.go:822: 4m25s of 5m0s: 692577 executions, 33392 successful
    rsg_test.go:822: 4m30s of 5m0s: 705471 executions, 33970 successful
    rsg_test.go:822: 4m35s of 5m0s: 719558 executions, 34624 successful
    rsg_test.go:822: 4m40s of 5m0s: 733014 executions, 35301 successful
    rsg_test.go:822: 4m45s of 5m0s: 746758 executions, 35906 successful
    rsg_test.go:822: 4m50s of 5m0s: 760650 executions, 36476 successful
    rsg_test.go:822: 4m55s of 5m0s: 775069 executions, 37075 successful
    rsg_test.go:858: 788706 executions, 37640 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO ( 'string' ) WITH DETACHED = FALSE", formattedSQL="BACKUP INTO 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration1285638548
--- FAIL: TestRandomSyntaxGeneration (301.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8138870?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8138870?buildTab=artifacts#/) on master @ [250238cd29102391dddbc8cc71380090c49ce509](https://github.com/cockroachdb/cockroach/commits/250238cd29102391dddbc8cc71380090c49ce509):


```
    rsg_test.go:822: 1m15s of 5m0s: 223496 executions, 12456 successful
    rsg_test.go:822: 1m20s of 5m0s: 239344 executions, 13275 successful
    rsg_test.go:822: 1m25s of 5m0s: 254364 executions, 14031 successful
    rsg_test.go:822: 1m30s of 5m0s: 269628 executions, 14783 successful
    rsg_test.go:822: 1m35s of 5m0s: 285015 executions, 15535 successful
    rsg_test.go:822: 1m40s of 5m0s: 300373 executions, 16272 successful
    rsg_test.go:822: 1m45s of 5m0s: 316134 executions, 17074 successful
    rsg_test.go:822: 1m50s of 5m0s: 330610 executions, 17798 successful
    rsg_test.go:822: 1m55s of 5m0s: 344801 executions, 18512 successful
    rsg_test.go:822: 2m0s of 5m0s: 359624 executions, 19268 successful
    rsg_test.go:822: 2m5s of 5m0s: 374802 executions, 19964 successful
    rsg_test.go:822: 2m10s of 5m0s: 389599 executions, 20679 successful
    rsg_test.go:822: 2m15s of 5m0s: 404513 executions, 21361 successful
    rsg_test.go:822: 2m20s of 5m0s: 419827 executions, 22099 successful
    rsg_test.go:822: 2m25s of 5m0s: 434766 executions, 22814 successful
    rsg_test.go:822: 2m30s of 5m0s: 450575 executions, 23520 successful
    rsg_test.go:822: 2m35s of 5m0s: 465557 executions, 24246 successful
    rsg_test.go:822: 2m40s of 5m0s: 480530 executions, 24940 successful
    rsg_test.go:822: 2m45s of 5m0s: 495717 executions, 25727 successful
    rsg_test.go:822: 2m50s of 5m0s: 511488 executions, 26456 successful
    rsg_test.go:822: 2m55s of 5m0s: 527091 executions, 27207 successful
    rsg_test.go:822: 3m0s of 5m0s: 542982 executions, 27932 successful
    rsg_test.go:822: 3m5s of 5m0s: 559024 executions, 28641 successful
    rsg_test.go:822: 3m10s of 5m0s: 574649 executions, 29336 successful
    rsg_test.go:822: 3m15s of 5m0s: 590896 executions, 30028 successful
    rsg_test.go:822: 3m20s of 5m0s: 606944 executions, 30746 successful
    rsg_test.go:822: 3m25s of 5m0s: 621807 executions, 31486 successful
    rsg_test.go:822: 3m30s of 5m0s: 636701 executions, 32114 successful
    rsg_test.go:822: 3m35s of 5m0s: 652185 executions, 32731 successful
    rsg_test.go:822: 3m40s of 5m0s: 667335 executions, 33443 successful
    rsg_test.go:822: 3m45s of 5m0s: 682317 executions, 34124 successful
    rsg_test.go:822: 3m50s of 5m0s: 697411 executions, 34826 successful
    rsg_test.go:822: 3m55s of 5m0s: 713148 executions, 35508 successful
    rsg_test.go:822: 4m0s of 5m0s: 728906 executions, 36203 successful
    rsg_test.go:822: 4m5s of 5m0s: 744029 executions, 36930 successful
    rsg_test.go:822: 4m10s of 5m0s: 758702 executions, 37579 successful
    rsg_test.go:822: 4m15s of 5m0s: 773099 executions, 38191 successful
    rsg_test.go:822: 4m20s of 5m0s: 787864 executions, 38803 successful
    rsg_test.go:822: 4m25s of 5m0s: 803260 executions, 39398 successful
    rsg_test.go:822: 4m30s of 5m0s: 817793 executions, 39995 successful
    rsg_test.go:822: 4m35s of 5m0s: 831827 executions, 40610 successful
    rsg_test.go:822: 4m40s of 5m0s: 845709 executions, 41198 successful
    rsg_test.go:822: 4m45s of 5m0s: 858719 executions, 41768 successful
    rsg_test.go:822: 4m50s of 5m0s: 872149 executions, 42304 successful
    rsg_test.go:822: 4m55s of 5m0s: 885392 executions, 42832 successful
    rsg_test.go:858: 898903 executions, 43369 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP TO 'string' INCREMENTAL FROM PLACEHOLDER , 'string' , PLACEHOLDER WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TO 'string' INCREMENTAL FROM 'placeholder', 'string', 'placeholder' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration3239795523
--- FAIL: TestRandomSyntaxGeneration (301.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8147266?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8147266?buildTab=artifacts#/) on master @ [0725273ac7f789ba8ed78aacaf73cc953ca47fe8](https://github.com/cockroachdb/cockroach/commits/0725273ac7f789ba8ed78aacaf73cc953ca47fe8):


```
    rsg_test.go:822: 1m15s of 5m0s: 214922 executions, 11739 successful
    rsg_test.go:822: 1m20s of 5m0s: 230090 executions, 12508 successful
    rsg_test.go:822: 1m25s of 5m0s: 245613 executions, 13263 successful
    rsg_test.go:822: 1m30s of 5m0s: 260624 executions, 13927 successful
    rsg_test.go:822: 1m35s of 5m0s: 275358 executions, 14602 successful
    rsg_test.go:822: 1m40s of 5m0s: 290843 executions, 15344 successful
    rsg_test.go:822: 1m45s of 5m0s: 305422 executions, 16024 successful
    rsg_test.go:822: 1m50s of 5m0s: 319916 executions, 16670 successful
    rsg_test.go:822: 1m55s of 5m0s: 334147 executions, 17385 successful
    rsg_test.go:822: 2m0s of 5m0s: 349110 executions, 18136 successful
    rsg_test.go:822: 2m5s of 5m0s: 364366 executions, 18839 successful
    rsg_test.go:822: 2m10s of 5m0s: 379881 executions, 19576 successful
    rsg_test.go:822: 2m15s of 5m0s: 395047 executions, 20273 successful
    rsg_test.go:822: 2m20s of 5m0s: 410166 executions, 21025 successful
    rsg_test.go:822: 2m25s of 5m0s: 425378 executions, 21799 successful
    rsg_test.go:822: 2m30s of 5m0s: 439439 executions, 22468 successful
    rsg_test.go:822: 2m35s of 5m0s: 454920 executions, 23144 successful
    rsg_test.go:822: 2m40s of 5m0s: 470691 executions, 23852 successful
    rsg_test.go:822: 2m45s of 5m0s: 486237 executions, 24610 successful
    rsg_test.go:822: 2m50s of 5m0s: 501871 executions, 25336 successful
    rsg_test.go:822: 2m55s of 5m0s: 517311 executions, 26070 successful
    rsg_test.go:822: 3m0s of 5m0s: 533078 executions, 26744 successful
    rsg_test.go:822: 3m5s of 5m0s: 548611 executions, 27490 successful
    rsg_test.go:822: 3m10s of 5m0s: 563661 executions, 28193 successful
    rsg_test.go:822: 3m15s of 5m0s: 578337 executions, 28900 successful
    rsg_test.go:822: 3m20s of 5m0s: 593250 executions, 29616 successful
    rsg_test.go:822: 3m25s of 5m0s: 608440 executions, 30321 successful
    rsg_test.go:822: 3m30s of 5m0s: 623449 executions, 31037 successful
    rsg_test.go:822: 3m35s of 5m0s: 637745 executions, 31742 successful
    rsg_test.go:822: 3m40s of 5m0s: 652137 executions, 32398 successful
    rsg_test.go:822: 3m45s of 5m0s: 667222 executions, 33127 successful
    rsg_test.go:822: 3m50s of 5m0s: 681986 executions, 33890 successful
    rsg_test.go:822: 3m55s of 5m0s: 697467 executions, 34596 successful
    rsg_test.go:822: 4m0s of 5m0s: 712765 executions, 35253 successful
    rsg_test.go:822: 4m5s of 5m0s: 727914 executions, 35937 successful
    rsg_test.go:822: 4m10s of 5m0s: 743397 executions, 36656 successful
    rsg_test.go:822: 4m15s of 5m0s: 758088 executions, 37349 successful
    rsg_test.go:822: 4m20s of 5m0s: 773568 executions, 38065 successful
    rsg_test.go:822: 4m25s of 5m0s: 789738 executions, 38774 successful
    rsg_test.go:822: 4m30s of 5m0s: 805076 executions, 39454 successful
    rsg_test.go:822: 4m35s of 5m0s: 820911 executions, 40188 successful
    rsg_test.go:822: 4m40s of 5m0s: 835635 executions, 40933 successful
    rsg_test.go:822: 4m45s of 5m0s: 850836 executions, 41653 successful
    rsg_test.go:822: 4m50s of 5m0s: 865645 executions, 42365 successful
    rsg_test.go:822: 4m55s of 5m0s: 880323 executions, 43066 successful
    rsg_test.go:858: 894382 executions, 43655 successful
    rsg_test.go:866: cannot parse output of Format: sql="PREPARE WORK AS BACKUP INTO LATEST IN ( 'string' ) WITH DETACHED = FALSE", formattedSQL="PREPARE \"work\" AS BACKUP INTO LATEST IN 'string' WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2905687841
--- FAIL: TestRandomSyntaxGeneration (300.43s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8153415?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8153415?buildTab=artifacts#/) on master @ [0725273ac7f789ba8ed78aacaf73cc953ca47fe8](https://github.com/cockroachdb/cockroach/commits/0725273ac7f789ba8ed78aacaf73cc953ca47fe8):


```
    rsg_test.go:822: 1m20s of 5m0s: 220767 executions, 11877 successful
    rsg_test.go:822: 1m25s of 5m0s: 235436 executions, 12466 successful
    rsg_test.go:822: 1m30s of 5m0s: 249549 executions, 13051 successful
    rsg_test.go:822: 1m35s of 5m0s: 262985 executions, 13643 successful
    rsg_test.go:822: 1m40s of 5m0s: 277018 executions, 14289 successful
    rsg_test.go:822: 1m45s of 5m0s: 291697 executions, 14984 successful
    rsg_test.go:822: 1m50s of 5m0s: 306482 executions, 15668 successful
    rsg_test.go:822: 1m55s of 5m0s: 321861 executions, 16411 successful
    rsg_test.go:822: 2m0s of 5m0s: 337009 executions, 17101 successful
    rsg_test.go:822: 2m5s of 5m0s: 351886 executions, 17808 successful
    rsg_test.go:822: 2m10s of 5m0s: 366264 executions, 18510 successful
    rsg_test.go:822: 2m15s of 5m0s: 380664 executions, 19229 successful
    rsg_test.go:822: 2m20s of 5m0s: 396109 executions, 19920 successful
    rsg_test.go:822: 2m25s of 5m0s: 410279 executions, 20614 successful
    rsg_test.go:822: 2m30s of 5m0s: 424752 executions, 21332 successful
    rsg_test.go:822: 2m35s of 5m0s: 438668 executions, 22006 successful
    rsg_test.go:822: 2m40s of 5m0s: 453105 executions, 22691 successful
    rsg_test.go:822: 2m45s of 5m0s: 467177 executions, 23287 successful
    rsg_test.go:822: 2m50s of 5m0s: 481436 executions, 23814 successful
    rsg_test.go:822: 2m55s of 5m0s: 495799 executions, 24399 successful
    rsg_test.go:822: 3m0s of 5m0s: 509622 executions, 24952 successful
    rsg_test.go:822: 3m5s of 5m0s: 523881 executions, 25518 successful
    rsg_test.go:822: 3m10s of 5m0s: 538141 executions, 26106 successful
    rsg_test.go:822: 3m15s of 5m0s: 552439 executions, 26677 successful
    rsg_test.go:822: 3m20s of 5m0s: 566162 executions, 27222 successful
    rsg_test.go:822: 3m25s of 5m0s: 579663 executions, 27749 successful
    rsg_test.go:822: 3m30s of 5m0s: 592172 executions, 28273 successful
    rsg_test.go:822: 3m35s of 5m0s: 604561 executions, 28775 successful
    rsg_test.go:822: 3m40s of 5m0s: 617045 executions, 29286 successful
    rsg_test.go:822: 3m45s of 5m0s: 629516 executions, 29790 successful
    rsg_test.go:822: 3m50s of 5m0s: 641713 executions, 30307 successful
    rsg_test.go:822: 3m55s of 5m0s: 653989 executions, 30791 successful
    rsg_test.go:822: 4m0s of 5m0s: 666618 executions, 31237 successful
    rsg_test.go:822: 4m5s of 5m0s: 678864 executions, 31732 successful
    rsg_test.go:822: 4m10s of 5m0s: 691364 executions, 32213 successful
    rsg_test.go:822: 4m15s of 5m0s: 703075 executions, 32676 successful
    rsg_test.go:822: 4m20s of 5m0s: 714744 executions, 33144 successful
    rsg_test.go:822: 4m25s of 5m0s: 727830 executions, 33632 successful
    rsg_test.go:822: 4m30s of 5m0s: 740258 executions, 34108 successful
    rsg_test.go:822: 4m35s of 5m0s: 751179 executions, 34543 successful
    rsg_test.go:822: 4m40s of 5m0s: 763273 executions, 35093 successful
    rsg_test.go:822: 4m45s of 5m0s: 775942 executions, 35614 successful
    rsg_test.go:822: 4m50s of 5m0s: 788943 executions, 36178 successful
    rsg_test.go:822: 4m55s of 5m0s: 800223 executions, 36623 successful
    rsg_test.go:822: 5m0s of 5m0s: 811547 executions, 37120 successful
    rsg_test.go:858: 811549 executions, 37121 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP CONVERSION . SKIP_MISSING_FOREIGN_KEYS . * INTO 'string' IN ( PLACEHOLDER , 'string' , FAMILY ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP TABLE conversion.skip_missing_foreign_keys.* INTO 'string' IN ('placeholder', 'string', 'family') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration604640137
--- FAIL: TestRandomSyntaxGeneration (301.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8159338?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8159338?buildTab=artifacts#/) on master @ [0725273ac7f789ba8ed78aacaf73cc953ca47fe8](https://github.com/cockroachdb/cockroach/commits/0725273ac7f789ba8ed78aacaf73cc953ca47fe8):


```
    rsg_test.go:822: 1m15s of 5m0s: 193540 executions, 10179 successful
    rsg_test.go:822: 1m20s of 5m0s: 208016 executions, 10908 successful
    rsg_test.go:822: 1m25s of 5m0s: 222002 executions, 11607 successful
    rsg_test.go:822: 1m30s of 5m0s: 235330 executions, 12275 successful
    rsg_test.go:822: 1m35s of 5m0s: 249117 executions, 12958 successful
    rsg_test.go:822: 1m40s of 5m0s: 263069 executions, 13694 successful
    rsg_test.go:822: 1m45s of 5m0s: 277631 executions, 14404 successful
    rsg_test.go:822: 1m50s of 5m0s: 291898 executions, 15144 successful
    rsg_test.go:822: 1m55s of 5m0s: 306514 executions, 15861 successful
    rsg_test.go:822: 2m0s of 5m0s: 320943 executions, 16596 successful
    rsg_test.go:822: 2m5s of 5m0s: 335671 executions, 17308 successful
    rsg_test.go:822: 2m10s of 5m0s: 350382 executions, 17975 successful
    rsg_test.go:822: 2m15s of 5m0s: 364805 executions, 18622 successful
    rsg_test.go:822: 2m20s of 5m0s: 378285 executions, 19252 successful
    rsg_test.go:822: 2m25s of 5m0s: 392747 executions, 19890 successful
    rsg_test.go:822: 2m30s of 5m0s: 406753 executions, 20507 successful
    rsg_test.go:822: 2m35s of 5m0s: 420661 executions, 21120 successful
    rsg_test.go:822: 2m40s of 5m0s: 434076 executions, 21658 successful
    rsg_test.go:822: 2m45s of 5m0s: 448221 executions, 22215 successful
    rsg_test.go:822: 2m50s of 5m0s: 462196 executions, 22768 successful
    rsg_test.go:822: 2m55s of 5m0s: 476012 executions, 23359 successful
    rsg_test.go:822: 3m0s of 5m0s: 490540 executions, 23877 successful
    rsg_test.go:822: 3m5s of 5m0s: 504656 executions, 24409 successful
    rsg_test.go:822: 3m10s of 5m0s: 519086 executions, 24980 successful
    rsg_test.go:822: 3m15s of 5m0s: 531722 executions, 25501 successful
    rsg_test.go:822: 3m20s of 5m0s: 544941 executions, 26045 successful
    rsg_test.go:822: 3m25s of 5m0s: 558473 executions, 26628 successful
    rsg_test.go:822: 3m30s of 5m0s: 571767 executions, 27226 successful
    rsg_test.go:822: 3m35s of 5m0s: 585830 executions, 27801 successful
    rsg_test.go:822: 3m40s of 5m0s: 598668 executions, 28330 successful
    rsg_test.go:822: 3m45s of 5m0s: 611253 executions, 28890 successful
    rsg_test.go:822: 3m50s of 5m0s: 624405 executions, 29392 successful
    rsg_test.go:822: 3m55s of 5m0s: 637842 executions, 29932 successful
    rsg_test.go:822: 4m0s of 5m0s: 651002 executions, 30475 successful
    rsg_test.go:822: 4m5s of 5m0s: 664296 executions, 30975 successful
    rsg_test.go:822: 4m10s of 5m0s: 676950 executions, 31493 successful
    rsg_test.go:822: 4m15s of 5m0s: 689796 executions, 32054 successful
    rsg_test.go:822: 4m20s of 5m0s: 702278 executions, 32556 successful
    rsg_test.go:822: 4m25s of 5m0s: 716920 executions, 33232 successful
    rsg_test.go:822: 4m30s of 5m0s: 731605 executions, 33891 successful
    rsg_test.go:822: 4m35s of 5m0s: 746725 executions, 34655 successful
    rsg_test.go:822: 4m40s of 5m0s: 760810 executions, 35347 successful
    rsg_test.go:822: 4m45s of 5m0s: 774803 executions, 36023 successful
    rsg_test.go:822: 4m50s of 5m0s: 788161 executions, 36690 successful
    rsg_test.go:822: 4m55s of 5m0s: 801891 executions, 37279 successful
    rsg_test.go:858: 815178 executions, 37860 successful
    rsg_test.go:866: cannot parse output of Format: sql="BACKUP INTO ( PLACEHOLDER , ident ) WITH OPTIONS ( DETACHED = FALSE )", formattedSQL="BACKUP INTO ('placeholder', 'ident') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration2601114815
--- FAIL: TestRandomSyntaxGeneration (300.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxGeneration [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8166115?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8166115?buildTab=artifacts#/) on master @ [1d7bd69205c2197ccac33df9e2e6d4ff8c0fdbcf](https://github.com/cockroachdb/cockroach/commits/1d7bd69205c2197ccac33df9e2e6d4ff8c0fdbcf):


```
    rsg_test.go:822: 1m20s of 5m0s: 209481 executions, 10389 successful
    rsg_test.go:822: 1m25s of 5m0s: 222911 executions, 10959 successful
    rsg_test.go:822: 1m30s of 5m0s: 236473 executions, 11505 successful
    rsg_test.go:822: 1m35s of 5m0s: 248541 executions, 11991 successful
    rsg_test.go:822: 1m40s of 5m0s: 260718 executions, 12498 successful
    rsg_test.go:822: 1m45s of 5m0s: 272860 executions, 13020 successful
    rsg_test.go:822: 1m50s of 5m0s: 286959 executions, 13737 successful
    rsg_test.go:822: 1m55s of 5m0s: 301494 executions, 14417 successful
    rsg_test.go:822: 2m0s of 5m0s: 315891 executions, 15137 successful
    rsg_test.go:822: 2m5s of 5m0s: 330886 executions, 15844 successful
    rsg_test.go:822: 2m10s of 5m0s: 345131 executions, 16487 successful
    rsg_test.go:822: 2m15s of 5m0s: 359216 executions, 17224 successful
    rsg_test.go:822: 2m20s of 5m0s: 373121 executions, 17889 successful
    rsg_test.go:822: 2m25s of 5m0s: 386843 executions, 18545 successful
    rsg_test.go:822: 2m30s of 5m0s: 401029 executions, 19251 successful
    rsg_test.go:822: 2m35s of 5m0s: 414824 executions, 19929 successful
    rsg_test.go:822: 2m40s of 5m0s: 427883 executions, 20527 successful
    rsg_test.go:822: 2m45s of 5m0s: 441145 executions, 21186 successful
    rsg_test.go:822: 2m50s of 5m0s: 454890 executions, 21794 successful
    rsg_test.go:822: 2m55s of 5m0s: 468888 executions, 22453 successful
    rsg_test.go:822: 3m0s of 5m0s: 483282 executions, 23089 successful
    rsg_test.go:822: 3m5s of 5m0s: 497396 executions, 23735 successful
    rsg_test.go:822: 3m10s of 5m0s: 510717 executions, 24386 successful
    rsg_test.go:822: 3m15s of 5m0s: 524560 executions, 25064 successful
    rsg_test.go:822: 3m20s of 5m0s: 539010 executions, 25712 successful
    rsg_test.go:822: 3m25s of 5m0s: 553338 executions, 26370 successful
    rsg_test.go:822: 3m30s of 5m0s: 567629 executions, 26985 successful
    rsg_test.go:822: 3m35s of 5m0s: 582434 executions, 27667 successful
    rsg_test.go:822: 3m40s of 5m0s: 596395 executions, 28337 successful
    rsg_test.go:822: 3m45s of 5m0s: 610610 executions, 28974 successful
    rsg_test.go:822: 3m50s of 5m0s: 624673 executions, 29640 successful
    rsg_test.go:822: 3m55s of 5m0s: 638573 executions, 30263 successful
    rsg_test.go:822: 4m0s of 5m0s: 651961 executions, 30933 successful
    rsg_test.go:822: 4m5s of 5m0s: 665156 executions, 31562 successful
    rsg_test.go:822: 4m10s of 5m0s: 679025 executions, 32191 successful
    rsg_test.go:822: 4m15s of 5m0s: 692361 executions, 32799 successful
    rsg_test.go:822: 4m20s of 5m0s: 706350 executions, 33374 successful
    rsg_test.go:822: 4m25s of 5m0s: 719699 executions, 33995 successful
    rsg_test.go:822: 4m30s of 5m0s: 733056 executions, 34625 successful
    rsg_test.go:822: 4m35s of 5m0s: 746554 executions, 35197 successful
    rsg_test.go:822: 4m40s of 5m0s: 760139 executions, 35840 successful
    rsg_test.go:822: 4m45s of 5m0s: 773212 executions, 36469 successful
    rsg_test.go:822: 4m50s of 5m0s: 786657 executions, 37117 successful
    rsg_test.go:822: 4m55s of 5m0s: 800287 executions, 37681 successful
    rsg_test.go:822: 5m0s of 5m0s: 812926 executions, 38285 successful
    rsg_test.go:858: 812928 executions, 38285 successful
    rsg_test.go:866: cannot parse output of Format: sql="PREPARE LEAST AS BACKUP CHAR INTO ( TIME , RELOCATE , PLACEHOLDER ) WITH DETACHED = FALSE", formattedSQL="PREPARE \"least\" AS BACKUP TABLE \"char\" INTO ('time', 'relocate', 'placeholder') WITH ": at or near "EOF": syntax error
    rsg_test.go:305: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxGeneration1158434932
--- FAIL: TestRandomSyntaxGeneration (301.45s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89363 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87572 sql/tests: TestRandomSyntaxGeneration failed [DROP OWNED BY timeout] [C-test-failure O-robot T-sql-schema branch-release-22.2]
- #77893 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot T-sql-sessions branch-release-22.1]
- #74271 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.2]
- #65210 sql/tests: TestRandomSyntaxGeneration failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxGeneration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/parser/testdata/backup_restore
+++ b/pkg/sql/parser/testdata/backup_restore
@@ -346,6 +346,16 @@ BACKUP TABLE (foo) TO ('bar') WITH revision_history = (true), detached, kms = ((
 BACKUP TABLE foo TO '_' WITH revision_history = _, detached, kms = ('_', '_') -- literals removed
 BACKUP TABLE _ TO 'bar' WITH revision_history = true, detached, kms = ('foo', 'bar') -- identifiers removed
 
+
+# Regression test for #95235.
+parse
+BACKUP foo TO 'bar' WITH OPTIONS (detached = false)
+----
+BACKUP TABLE foo TO 'bar' WITH detached = FALSE -- normalized!
+BACKUP TABLE (foo) TO ('bar') WITH detached = FALSE -- fully parenthesized
+BACKUP TABLE foo TO '_' WITH detached = FALSE -- literals removed
+BACKUP TABLE _ TO 'bar' WITH detached = FALSE -- identifiers removed
+
 parse
 BACKUP TENANT 36 TO 'bar'
 ----

--- a/pkg/sql/sem/tree/backup.go
+++ b/pkg/sql/sem/tree/backup.go
@@ -272,9 +272,12 @@ func (o *BackupOptions) Format(ctx *FmtCtx) {
 		}
 	}
 
-	if o.Detached == DBoolTrue {
+	if o.Detached != nil {
 		maybeAddSep()
 		ctx.WriteString("detached")
+		if o.Detached != DBoolTrue {
+			ctx.WriteString(" = FALSE")
+		}
 	}
 
 	if o.EncryptionKMSURI != nil {
@@ -334,7 +337,8 @@ func (o *BackupOptions) CombineWith(other *BackupOptions) error {
 func (o BackupOptions) IsDefault() bool {
 	options := BackupOptions{}
 	return o.CaptureRevisionHistory == options.CaptureRevisionHistory &&
-		o.Detached == options.Detached && cmp.Equal(o.EncryptionKMSURI, options.EncryptionKMSURI) &&
+		o.Detached == options.Detached &&
+		cmp.Equal(o.EncryptionKMSURI, options.EncryptionKMSURI) &&
 		o.EncryptionPassphrase == options.EncryptionPassphrase &&
 		cmp.Equal(o.IncrementalStorage, options.IncrementalStorage)
 }

